<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for GenomicFeatures on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><B>[<A href="..#G">G</A>]</B></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-06 11:00:32 -0000 (Tue, 06 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for GenomicFeatures on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GenomicFeatures package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GenomicFeatures.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard timeout">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>795</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GenomicFeatures</A>&nbsp;1.53.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/GenomicFeatures">landing page</A>)</SPAN><BR>H. Pagès<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-05&nbsp;06:35:06&nbsp;-0000&nbsp;(Mon,&nbsp;05&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GenomicFeatures</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">2f58fb7</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;13:49:30&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GenomicFeatures</TD></TR>
<TR><TD><B>Version</B>: 1.53.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:GenomicFeatures.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings GenomicFeatures_1.53.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-05 22:57:09 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-05 23:37:09 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 2400.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: None</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GenomicFeatures.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:GenomicFeatures.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings GenomicFeatures_1.53.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/GenomicFeatures.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘GenomicFeatures/DESCRIPTION’ ... OK
* this is package ‘GenomicFeatures’ version ‘1.53.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;BiocGenerics&#x27;, &#x27;S4Vectors&#x27;, &#x27;IRanges&#x27;, &#x27;GenomeInfoDb&#x27;,
  &#x27;GenomicRanges&#x27;, &#x27;AnnotationDbi&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GenomicFeatures’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
&#x27;:::&#x27; call which should be &#x27;::&#x27;: ‘rtracklayer:::tableNames’
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘AnnotationDbi:::.getMetaValue’ ‘AnnotationDbi:::.valid.colnames’
  ‘AnnotationDbi:::.valid.metadata.table’
  ‘AnnotationDbi:::.valid.table.colnames’ ‘AnnotationDbi:::dbEasyQuery’
  ‘AnnotationDbi:::dbQuery’ ‘AnnotationDbi:::smartKeys’
  ‘BiocGenerics:::testPackage’ ‘GenomeInfoDb:::check_tax_id’
  ‘GenomeInfoDb:::getSeqlevelsReplacementMode’
  ‘GenomeInfoDb:::lookup_organism_by_tax_id’
  ‘GenomeInfoDb:::lookup_tax_id_by_organism’
  ‘GenomeInfoDb:::make_circ_flags_from_circ_seqs’
  ‘GenomeInfoDb:::normarg_new2old’
  ‘GenomicRanges:::unsafe.transcriptLocs2refLocs’
  ‘GenomicRanges:::unsafe.transcriptWidths’
  ‘IRanges:::regroupBySupergroup’ ‘S4Vectors:::V_recycle’
  ‘S4Vectors:::anyMissingOrOutside’ ‘S4Vectors:::decodeRle’
  ‘S4Vectors:::extract_data_frame_rows’ ‘S4Vectors:::quick_togroup’
  ‘biomaRt:::martBM’ ‘biomaRt:::martDataset’ ‘biomaRt:::martHost’
  ‘rtracklayer:::resourceDescription’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) FeatureDb-class.Rd:31-34: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:30-33: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:34-39: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:40-43: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:44-52: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:53-59: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:60-65: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:66-71: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:72-80: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:21-25: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:26-50: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:51-55: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:56-62: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘GenomicFeatures-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: makeTxDbFromBiomart
&gt; ### Title: Make a TxDb object from annotations available on a BioMart
&gt; ###   database
&gt; ### Aliases: makeTxDbFromBiomart getChromInfoFromBiomart
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## A. BASIC USAGE
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; ## We can use listDatasets() from the biomaRt package to list the
&gt; ## datasets available in the &quot;ENSEMBL_MART_ENSEMBL&quot; BioMart database:
&gt; library(biomaRt)
&gt; listMarts(host=&quot;https://www.ensembl.org&quot;)
               biomart                version
1 ENSEMBL_MART_ENSEMBL      Ensembl Genes 109
2   ENSEMBL_MART_MOUSE      Mouse strains 109
3     ENSEMBL_MART_SNP  Ensembl Variation 109
4 ENSEMBL_MART_FUNCGEN Ensembl Regulation 109
&gt; mart &lt;- useEnsembl(biomart=&quot;ENSEMBL_MART_ENSEMBL&quot;, host=&quot;https://www.ensembl.org&quot;)
&gt; datasets &lt;- listDatasets(mart)
&gt; head(datasets)
                       dataset                           description
1 abrachyrhynchus_gene_ensembl Pink-footed goose genes (ASM259213v1)
2     acalliptera_gene_ensembl      Eastern happy genes (fAstCal1.2)
3   acarolinensis_gene_ensembl       Green anole genes (AnoCar2.0v2)
4    acchrysaetos_gene_ensembl       Golden eagle genes (bAquChr1.2)
5    acitrinellus_gene_ensembl        Midas cichlid genes (Midas_v5)
6    amelanoleuca_gene_ensembl       Giant panda genes (ASM200744v2)
      version
1 ASM259213v1
2  fAstCal1.2
3 AnoCar2.0v2
4  bAquChr1.2
5    Midas_v5
6 ASM200744v2
&gt; subset(datasets, grepl(&quot;elegans&quot;, dataset, ignore.case=TRUE))
                 dataset                                          description
27 celegans_gene_ensembl Caenorhabditis elegans (PRJNA13758) genes (WBcel235)
    version
27 WBcel235
&gt; 
&gt; ## Retrieve the full transcript dataset for Worm:
&gt; txdb1 &lt;- makeTxDbFromBiomart(dataset=&quot;celegans_gene_ensembl&quot;)
Download and preprocess the &#x27;transcripts&#x27; data frame ... OK
Download and preprocess the &#x27;chrominfo&#x27; data frame ... FAILED! (=&gt; skipped)
Download and preprocess the &#x27;splicings&#x27; data frame ... Error in curl::curl_fetch_memory(url, handle = handle) : 
  Timeout was reached: [www.ensembl.org:443] Operation timed out after 300000 milliseconds with 14952778 bytes received
Calls: makeTxDbFromBiomart ... request_fetch -&gt; request_fetch.write_memory -&gt; &lt;Anonymous&gt;
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘run_unitTests.R’</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GenomicFeatures.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL GenomicFeatures
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘GenomicFeatures’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GenomicFeatures)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">GenomicFeatures.Rcheck/tests/run_unitTests.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;GenomicFeatures&quot;) || stop(&quot;unable to load GenomicFeatures package&quot;)
Loading required package: GenomicFeatures
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicRanges
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

[1] TRUE
&gt; GenomicFeatures:::.test()
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer
Download and preprocess the &#x27;transcripts&#x27; data frame ... OK
Download and preprocess the &#x27;chrominfo&#x27; data frame ... OK
Download and preprocess the &#x27;splicings&#x27; data frame ... OK
Download and preprocess the &#x27;genes&#x27; data frame ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
Error in x$.self$finalize() : attempt to apply non-function
In addition: Warning messages:
1: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 3 out-of-bound ranges located on sequences a,
  b, and c. Note that ranges located on a sequence whose length is
  unknown (NA) or on a circular sequence are not considered out-of-bound
  (use seqlengths() and isCircular() to get the lengths and circularity
  flags of the underlying sequences). You can use trim() to trim these
  ranges. See ?`trim,GenomicRanges-method` for more information.
2: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 1 out-of-bound range located on sequence c.
  Note that ranges located on a sequence whose length is unknown (NA) or
  on a circular sequence are not considered out-of-bound (use
  seqlengths() and isCircular() to get the lengths and circularity flags
  of the underlying sequences). You can use trim() to trim these ranges.
  See ?`trim,GenomicRanges-method` for more information.
3: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 1 out-of-bound range located on sequence c.
  Note that ranges located on a sequence whose length is unknown (NA) or
  on a circular sequence are not considered out-of-bound (use
  seqlengths() and isCircular() to get the lengths and circularity flags
  of the underlying sequences). You can use trim() to trim these ranges.
  See ?`trim,GenomicRanges-method` for more information.
4: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 4 out-of-bound ranges located on sequences 1,
  2, 3, and 4. Note that ranges located on a sequence whose length is
  unknown (NA) or on a circular sequence are not considered out-of-bound
  (use seqlengths() and isCircular() to get the lengths and circularity
  flags of the underlying sequences). You can use trim() to trim these
  ranges. See ?`trim,GenomicRanges-method` for more information.
5: In .get_cds_IDX(mcols0$type, mcols0$phase) :
  The &quot;phase&quot; metadata column contains non-NA values for features of type
  exon. This information was ignored.
6: In .get_cds_IDX(mcols0$type, mcols0$phase) :
  The &quot;phase&quot; metadata column contains non-NA values for features of type
  stop_codon. This information was ignored.
7: In .get_cds_IDX(mcols0$type, mcols0$phase) :
  The &quot;phase&quot; metadata column contains non-NA values for features of type
  stop_codon. This information was ignored.
8: In (function (seqlevels, genome, new_style)  :
  cannot switch some hg19&#x27;s seqlevels from UCSC to NCBI style
Error in x$.self$finalize() : attempt to apply non-function
In addition: Warning message:
call dbDisconnect() when finished working with a connection 
Error in x$.self$finalize() : attempt to apply non-function
Error in x$.self$finalize() : attempt to apply non-function
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GenomicFeatures.Rcheck/GenomicFeatures-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>FeatureDb-class</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.074</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TxDb-class</TD><TD style="text-align: right;">0.715</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">0.810</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>as-format-methods</TD><TD style="text-align: right;">1.723</TD><TD style="text-align: right;">0.080</TD><TD style="text-align: right;">1.807</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coordinate-mapping-methods</TD><TD style="text-align: right;">51.780</TD><TD style="text-align: right;"> 1.690</TD><TD style="text-align: right;">61.714</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coverageByTranscript</TD><TD style="text-align: right;">108.900</TD><TD style="text-align: right;">  2.423</TD><TD style="text-align: right;">111.548</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exonicParts</TD><TD style="text-align: right;">50.587</TD><TD style="text-align: right;"> 0.339</TD><TD style="text-align: right;">51.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extendExonsIntoIntrons</TD><TD style="text-align: right;">3.773</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">4.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extractTranscriptSeqs</TD><TD style="text-align: right;">16.634</TD><TD style="text-align: right;"> 0.168</TD><TD style="text-align: right;">16.839</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extractUpstreamSeqs</TD><TD style="text-align: right;">1.724</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">1.784</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>features</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPromoterSeq-methods</TD><TD style="text-align: right;">0.707</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.720</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>id2name</TD><TD style="text-align: right;">0.256</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.257</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeFeatureDbFromUCSC</TD><TD style="text-align: right;"> 140.290</TD><TD style="text-align: right;">   7.315</TD><TD style="text-align: right;">1171.431</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxDb</TD><TD style="text-align: right;">0.144</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.144</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
