<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - BUILD results for breakpointR on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><B>[<A href="..#B">B</A>]</B></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-25 05:45:51 -0000 (Thu, 25 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4223</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">BUILD results for breakpointR on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the breakpointR package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/breakpointR.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>232</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc selected">BUILD</TD><TD class="STAGE checksrc">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">breakpointR</A>&nbsp;1.19.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/breakpointR">landing page</A>)</SPAN><BR>David Porubsky<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-24&nbsp;15:28:50&nbsp;-0000&nbsp;(Wed,&nbsp;24&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/breakpointR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">49b2e42</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:04:34&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc selected"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc"><SPAN class="glyph skipped">skipped</SPAN></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: breakpointR</TD></TR>
<TR><TD><B>Version</B>: 1.19.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data breakpointR</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-24 21:08:24 -0000 (Wed, 24 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-24 21:09:57 -0000 (Wed, 24 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 93.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>PackageFile</B>: None</TD></TR>
<TR><TD><B>PackageFileSize</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data breakpointR
###
##############################################################################
##############################################################################


* checking for file ‘breakpointR/DESCRIPTION’ ... OK
* preparing ‘breakpointR’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘breakpointR.Rnw’ using knitr
breakpointr            package:breakpointR             R Documentation

_M_a_i_n _f_u_n_c_t_i_o_n _f_o_r _t_h_e &#x27;_b_r_e_a_k_p_o_i_n_t_R&#x27; _p_a_c_k_a_g_e

_D_e_s_c_r_i_p_t_i_o_n:

     This function is an easy-to-use wrapper to find breakpoints with
     &#x27;runBreakpointr&#x27; in parallel, write the results to file, plot
     results and find hotspots.

_U_s_a_g_e:

     breakpointr(
       inputfolder,
       outputfolder,
       configfile = NULL,
       numCPU = 1,
       reuse.existing.files = FALSE,
       windowsize = 1e+06,
       binMethod = &quot;size&quot;,
       multi.sizes = NULL,
       pairedEndReads = FALSE,
       pair2frgm = FALSE,
       chromosomes = NULL,
       min.mapq = 10,
       filtAlt = FALSE,
       genoT = &quot;fisher&quot;,
       trim = 10,
       peakTh = 0.33,
       zlim = 3.291,
       background = 0.05,
       minReads = 10,
       maskRegions = NULL,
       callHotSpots = FALSE,
       conf = 0.99
     )
     
_A_r_g_u_m_e_n_t_s:

inputfolder: Folder with BAM files.

outputfolder: Folder to output the results. If it does not exist it
          will be created.

configfile: A file specifying the parameters of this function (without
          &#x27;inputfolder&#x27;, &#x27;outputfolder&#x27; and &#x27;configfile&#x27;). Having the
          parameters in a file can be handy if many samples with the
          same parameter settings are to be run. If a &#x27;configfile&#x27; is
          specified, it will take priority over the command line
          parameters.

  numCPU: The numbers of CPUs that are used. Should not be more than
          available on your machine.

reuse.existing.files: A logical indicating whether or not existing
          files in &#x27;outputfolder&#x27; should be reused.

windowsize: The window size used to calculate deltaWs, either number of
          reads or genomic size depending on &#x27;binMethod&#x27;.

binMethod: Method used to calculate optimal number of reads in the
          window (&quot;size&quot;, &quot;reads&quot;). By default &#x27;binMethod=&#x27;size&#x27;&#x27;.

multi.sizes: User defined multiplications of the original window size.

pairedEndReads: Set to &#x27;TRUE&#x27; if you have paired-end reads in your
          file.

pair2frgm: Set to &#x27;TRUE&#x27; if every paired-end read should be merged into
          a single fragment.

chromosomes: If only a subset of the chromosomes should be binned,
          specify them here.

min.mapq: Minimum mapping quality when importing from BAM files.

 filtAlt: Set to &#x27;TRUE&#x27; if you want to filter out alternative
          alignments defined in &#x27;XA&#x27; tag.

   genoT: A method (&#x27;fisher&#x27; or &#x27;binom&#x27;) to genotype regions defined by
          a set of breakpoints.

    trim: The amount of outliers in deltaWs removed to calculate the
          stdev (10 will remove top 10% and bottom 10% of deltaWs).

  peakTh: The treshold that the peak deltaWs must pass to be considered
          a breakpoint (e.g. 0.33 is 1/3 of max(deltaW)).

    zlim: The number of stdev that the deltaW must pass the peakTh
          (ensures only significantly higher peaks are considered).

background: The percent (e.g. 0.05 = 5%) of background reads allowed
          for WW or CC genotype calls.

minReads: The minimal number of reads between two breaks required for
          genotyping.

maskRegions: List of regions to be excluded from the analysis
          (tab-separated file: chromosomes start end).

callHotSpots: Search for regions of high abundance of breakpoints in
          single cells.

    conf: Desired confidence interval of localized breakpoints.

_V_a_l_u_e:

     &#x27;NULL&#x27;

_A_u_t_h_o_r(_s):

     David Porubsky, Aaron Taudt, Ashley Sanders

_E_x_a_m_p_l_e_s:

     ## Not run:
     
     ## The following call produces plots and genome browser files for all BAM files in &quot;my-data-folder&quot;
     breakpointr(inputfolder=&quot;my-data-folder&quot;, outputfolder=&quot;my-output-folder&quot;)
     ## End(Not run)
     

Error: processing vignette &#x27;breakpointR.Rnw&#x27; failed with diagnostics:
unable to run pdflatex on &#x27;breakpointR.tex&#x27;
LaTeX errors:
! LaTeX Error: File `framed.sty&#x27; not found.

Type X to quit or &lt;RETURN&gt; to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
&lt;read *&gt; 
         
l.137 \definecolor
                  {shadecolor}{RGB}{240,240,240}^^M
!  ==&gt; Fatal error occurred, no output PDF file produced!
--- failed re-building ‘breakpointR.Rnw’

SUMMARY: processing the following file failed:
  ‘breakpointR.Rnw’

Error: Vignette re-building failed.
Execution halted
</PRE>
</DIV></BODY>
</HTML>
