<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for sitadela on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><B>[<A href="..#S">S</A>]</B></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-05 06:03:08 -0000 (Mon, 05 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for sitadela on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the sitadela package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/sitadela.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard timeout">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1912</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">sitadela</A>&nbsp;1.9.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/sitadela">landing page</A>)</SPAN><BR>Panagiotis Moulos<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-03&nbsp;10:56:11&nbsp;-0000&nbsp;(Sat,&nbsp;03&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/sitadela</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">0320ace</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:27:34&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: sitadela</TD></TR>
<TR><TD><B>Version</B>: 1.9.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:sitadela.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings sitadela_1.9.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-05 00:09:21 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-05 00:49:21 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 2400.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: None</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: sitadela.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:sitadela.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings sitadela_1.9.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/sitadela.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘sitadela/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘sitadela’ version ‘1.9.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘sitadela’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking startup messages can be suppressed ... NOTE
sitadela database found at /home/biocbuild/.local/share/R/sitadela directory

It looks like this package (or a package it requires) has a startup
message which cannot be suppressed: see ?packageStartupMessage.
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                     user system elapsed
addCustomAnnotation 5.960  0.250   6.223
testFuns            4.342  0.395   7.696
getAnnotation       2.924  0.644 198.917
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">sitadela.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL sitadela
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘sitadela’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
sitadela database found at /home/biocbuild/.local/share/R/sitadela directory
** testing if installed package can be loaded from final location
sitadela database found at /home/biocbuild/.local/share/R/sitadela directory
** testing if installed package keeps a record of temporary installation path
* DONE (sitadela)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">sitadela.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;sitadela&quot;)
sitadela database found at /home/biocbuild/.local/share/R/sitadela directory
Scheduling 1 tests
==================================================


Running test 1 of 1 scheduled
Testing level gene from mm10 from latest version
Using Ensembl host https://www.ensembl.org
Test 1 successful!
Created 56916 features
Sample data:
                   chromosome   start     end            gene_id gc_content
ENSMUSG00000102693       chr1 3143476 3144545 ENSMUSG00000102693      34.21
ENSMUSG00000064842       chr1 3172239 3172348 ENSMUSG00000064842      36.36
ENSMUSG00000051951       chr1 3276124 3741721 ENSMUSG00000051951      38.51
ENSMUSG00000102851       chr1 3322980 3323459 ENSMUSG00000102851      39.79
ENSMUSG00000103377       chr1 3435954 3438772 ENSMUSG00000103377      40.79
ENSMUSG00000104017       chr1 3445779 3448011 ENSMUSG00000104017      36.99
                   strand     gene_name              biotype
ENSMUSG00000102693      + 4933401J01Rik                  TEC
ENSMUSG00000064842      +       Gm26206                snRNA
ENSMUSG00000051951      -          Xkr4       protein_coding
ENSMUSG00000102851      +       Gm18956 processed_pseudogene
ENSMUSG00000103377      -       Gm37180                  TEC
ENSMUSG00000104017      -       Gm37363                  TEC
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================


Running test 1 of 1 scheduled
Testing level transcript from mm10 from latest version
Using Ensembl host https://www.ensembl.org
Caught error: Error in curl::curl_fetch_memory(url, handle = handle): Timeout was reached: [www.ensembl.org:443] Operation timed out after 300000 milliseconds with 9677273 bytes received

This error is most probably related to biomaRt request timeouts! Using fallback solution...
Test 1 successful!
Created 149347 features
Sample data:
                   chromosome   start     end      transcript_id
ENSMUST00000193812       chr1 3143476 3144545 ENSMUST00000193812
ENSMUST00000082908       chr1 3172239 3172348 ENSMUST00000082908
ENSMUST00000162897       chr1 3276124 3286567 ENSMUST00000162897
ENSMUST00000159265       chr1 3276746 3285855 ENSMUST00000159265
ENSMUST00000070533       chr1 3284705 3741721 ENSMUST00000070533
ENSMUST00000192857       chr1 3322980 3323459 ENSMUST00000192857
                              gene_id strand     gene_name              biotype
ENSMUST00000193812 ENSMUSG00000102693      + 4933401J01Rik                  TEC
ENSMUST00000082908 ENSMUSG00000064842      +       Gm26206                snRNA
ENSMUST00000162897 ENSMUSG00000051951      -          Xkr4       protein_coding
ENSMUST00000159265 ENSMUSG00000051951      -          Xkr4       protein_coding
ENSMUST00000070533 ENSMUSG00000051951      -          Xkr4       protein_coding
ENSMUST00000192857 ENSMUSG00000102851      +       Gm18956 processed_pseudogene
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================


Running test 1 of 1 scheduled
Testing level gene from danrer11 from latest version
Using Ensembl host https://www.ensembl.org
Test 1 successful!
Created 32020 features
Sample data:
                   chromosome start   end            gene_id gc_content strand
ENSDARG00000099104       chr1  6408 12027 ENSDARG00000099104      43.42      -
ENSDARG00000102407       chr1 11822 16373 ENSDARG00000102407      54.39      +
ENSDARG00000102097       chr1 18716 23389 ENSDARG00000102097      52.70      +
ENSDARG00000099319       chr1 25585 27255 ENSDARG00000099319      37.64      +
ENSDARG00000099640       chr1 27690 34330 ENSDARG00000099640      53.02      +
ENSDARG00000104071       chr1 36552 39191 ENSDARG00000104071      50.64      +
                    gene_name        biotype
ENSDARG00000099104      rpl24 protein_coding
ENSDARG00000102407      cep97 protein_coding
ENSDARG00000102097     nfkbiz protein_coding
ENSDARG00000099319 CU651657.1 protein_coding
ENSDARG00000099640        eed protein_coding
ENSDARG00000104071    hikeshi protein_coding
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================


Running test 1 of 1 scheduled
Testing level transcript from danrer11 from latest version
Using Ensembl host https://www.ensembl.org
Test 1 successful!
Created 59253 features
Sample data:
                   chromosome start   end      transcript_id            gene_id
ENSDART00000164359       chr1  6408 12027 ENSDART00000164359 ENSDARG00000099104
ENSDART00000157701       chr1  6642  9919 ENSDART00000157701 ENSDARG00000099104
ENSDART00000158290       chr1  6642  7335 ENSDART00000158290 ENSDARG00000099104
ENSDART00000167898       chr1  6644 11725 ENSDART00000167898 ENSDARG00000099104
ENSDART00000166393       chr1 11822 16373 ENSDART00000166393 ENSDARG00000102407
ENSDART00000161842       chr1 11992 14058 ENSDART00000161842 ENSDARG00000102407
                   strand gene_name        biotype
ENSDART00000164359      -     rpl24 protein_coding
ENSDART00000157701      -     rpl24 protein_coding
ENSDART00000158290      -     rpl24 protein_coding
ENSDART00000167898      -     rpl24 protein_coding
ENSDART00000166393      +     cep97 protein_coding
ENSDART00000161842      +     cep97 protein_coding
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 2 tests
==================================================

Running test 1 of 2 scheduled
Opening sitadela SQLite database /tmp/RtmpNCHRBE/file27b6c67aa5bd64
  Importing GTF /home/biocbuild/R/R-4.3.0/site-library/sitadela/dummy.gtf.gz as GTF to make id map
  Making id map
  Importing GTF /home/biocbuild/R/R-4.3.0/site-library/sitadela/dummy.gtf.gz as TxDb
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Retrieving gene annotation for dummy from dummy_db version 1 from /home/biocbuild/R/R-4.3.0/site-library/sitadela/dummy.gtf.gz
Retrieving transcript annotation for dummy from dummy_db version 1
Retrieving summarized transcript annotation for dummy from dummy_db version 1
Retrieving 3&#x27; UTR annotation for dummy from dummy_db version 1
Retrieving summarized 3&#x27; UTR annotation per gene for dummy from dummy_db version 1
Retrieving summarized 3&#x27; UTR annotation per transcript for dummy from dummy_db version 1
Retrieving exon annotation for dummy from dummy_db version 1
Retrieving summarized exon annotation for dummy from dummy_db version 1
Retrieving extended exon annotation for dummy from dummy_db version 1
Retrieving summarized transcript exon annotation for dummy from dummy_db version 1
Test 1 successful!

Running test 2 of 2 scheduled
Test 2 successful!
Deleting temporary database /tmp/RtmpNCHRBE/file27b6c67aa5bd64
Scheduling 2 tests
==================================================

Running test 1 of 2 scheduled

********************************************************
This is sitadela 1.9.0 genomic region annotation builder
********************************************************
sitadela database found at /tmp/RtmpNCHRBE directory

========================================================
2023-06-05 00:39:21 - Try 1
========================================================

Opening sitadela SQLite database /tmp/RtmpNCHRBE/file27b6c65caef095
Retrieving genome information for dm6 from ensembl
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Retrieving gene annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Retrieving transcript annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Merging transcripts for dm6 from ensembl version 102
Retrieving 3&#x27; UTR annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Merging gene 3&#x27; UTRs for dm6 from ensembl version 102
Merging transcript 3&#x27; UTRs for dm6 from ensembl version 102
Retrieving exon annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Retrieving extended exon annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">sitadela.Rcheck/sitadela-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>addAnnotation</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addCustomAnnotation</TD><TD style="text-align: right;">5.960</TD><TD style="text-align: right;">0.250</TD><TD style="text-align: right;">6.223</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getAnnotation</TD><TD style="text-align: right;">  2.924</TD><TD style="text-align: right;">  0.644</TD><TD style="text-align: right;">198.917</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getInstalledAnnotations</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSeqInfo</TD><TD style="text-align: right;">0.208</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.858</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getUserAnnotations</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getsetDbPath</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>importCustomAnnotation</TD><TD style="text-align: right;">1.149</TD><TD style="text-align: right;">0.116</TD><TD style="text-align: right;">1.268</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadAnnotation</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>removeAnnotation</TD><TD style="text-align: right;">4.245</TD><TD style="text-align: right;">0.537</TD><TD style="text-align: right;">4.793</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>testFuns</TD><TD style="text-align: right;">4.342</TD><TD style="text-align: right;">0.395</TD><TD style="text-align: right;">7.696</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
