<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for EpiMix on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><B>[<A href="..#E">E</A>]</B></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-05 06:02:59 -0000 (Mon, 05 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for EpiMix on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the EpiMix package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/EpiMix.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>634</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">EpiMix</A>&nbsp;1.3.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/EpiMix">landing page</A>)</SPAN><BR>Yuanning Zheng<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-03&nbsp;10:56:11&nbsp;-0000&nbsp;(Sat,&nbsp;03&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/EpiMix</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">8a49eb8</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:39:07&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: EpiMix</TD></TR>
<TR><TD><B>Version</B>: 1.3.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:EpiMix.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings EpiMix_1.3.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-04 16:19:07 -0000 (Sun, 04 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-04 16:49:42 -0000 (Sun, 04 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1834.5 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: EpiMix.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:EpiMix.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings EpiMix_1.3.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/EpiMix.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘EpiMix/DESCRIPTION’ ... OK
* this is package ‘EpiMix’ version ‘1.3.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘EpiMix’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... NOTE
Problems with news in ‘NEWS.md’:
No news entries found.
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... WARNING
&#x27;::&#x27; or &#x27;:::&#x27; imports not declared from:
  &#x27;miRBaseConverter&#x27; &#x27;multiMiR&#x27;
&#x27;loadNamespace&#x27; or &#x27;requireNamespace&#x27; calls not declared from:
  &#x27;miRBaseConverter&#x27; &#x27;multiMiR&#x27;
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... WARNING
&#x27;library&#x27; or &#x27;require&#x27; calls not declared from:
  ‘miRBaseConverter’ ‘multiMiR’
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                            user system elapsed
Preprocess_DNAMethylation 38.301   2.01  52.561
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Methylation_Modeling.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
--- re-building ‘Methylation_Modeling.Rmd’ using rmarkdown
trying URL &#x27;https://egg2.wustl.edu/roadmap/data/byFileType/chromhmmSegmentations/ChmmModels/core_K27ac/jointModel/final/E096_18_core_K27ac_hg38lift_mnemonics.bed.gz&#x27;
Content type &#x27;application/x-gzip&#x27; length 3835377 bytes (3.7 MB)
==================================================
downloaded 3.7 MB

trying URL &#x27;https://gdac.broadinstitute.org/runs/stddata__latest/data/OV/20160128/gdac.broadinstitute.org_OV.Merge_methylation__humanmethylation27__jhu_usc_edu__Level_3__within_bioassay_data_set_function__data.Level_3.2016012800.0.0.tar.gz&#x27;
Content type &#x27;application/x-gzip&#x27; length 152968396 bytes (145.9 MB)
==================================================
downloaded 145.9 MB

trying URL &#x27;https://gdac.broadinstitute.org/runs/stddata__latest/data/OV/20160128/gdac.broadinstitute.org_OV.Merge_methylation__humanmethylation450__jhu_usc_edu__Level_3__within_bioassay_data_set_function__data.Level_3.2016012800.0.0.tar.gz&#x27;
Content type &#x27;application/x-gzip&#x27; length 51525820 bytes (49.1 MB)
==================================================
downloaded 49.1 MB

trying URL &#x27;https://gdac.broadinstitute.org/runs/stddata__latest/data/OV/20160128/gdac.broadinstitute.org_OV.Merge_transcriptome__agilentg4502a_07_3__unc_edu__Level_3__unc_lowess_normalization_gene_level__data.Level_3.2016012800.0.0.tar.gz&#x27;
Content type &#x27;application/x-gzip&#x27; length 36996192 bytes (35.3 MB)
==================================================
downloaded 35.3 MB

trying URL &#x27;https://gdac.broadinstitute.org/runs/stddata__latest/data/OV/20160128/gdac.broadinstitute.org_OV.miRseq_Preprocess.Level_3.2016012800.0.0.tar.gz&#x27;
Content type &#x27;application/x-gzip&#x27; length 2663096 bytes (2.5 MB)
==================================================
downloaded 2.5 MB

Warning in doTryCatch(return(expr), name, parentenv, handler) :
  &quot;offest&quot; is not a graphical parameter
trying URL &#x27;https://gdac.broadinstitute.org/runs/stddata__latest/data/LUAD/20160128/gdac.broadinstitute.org_LUAD.Merge_Clinical.Level_1.2016012800.0.0.tar.gz&#x27;
Content type &#x27;application/x-gzip&#x27; length 1777018 bytes (1.7 MB)
==================================================
downloaded 1.7 MB

trying URL &#x27;https://gdac.broadinstitute.org/runs/stddata__latest/data/LUAD/20160128/gdac.broadinstitute.org_LUAD.Merge_Clinical.Level_1.2016012800.0.0.tar.gz&#x27;
Content type &#x27;application/x-gzip&#x27; length 1777018 bytes (1.7 MB)
==================================================
downloaded 1.7 MB


Quitting from lines 579-586 [unnamed-chunk-40] (Methylation_Modeling.Rmd)
Error: processing vignette &#x27;Methylation_Modeling.Rmd&#x27; failed with diagnostics:
there is no package called &#x27;multiMiR&#x27;
--- failed re-building ‘Methylation_Modeling.Rmd’

SUMMARY: processing the following file failed:
  ‘Methylation_Modeling.Rmd’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 2 WARNINGs, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/EpiMix.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">EpiMix.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL EpiMix
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘EpiMix’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (EpiMix)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">EpiMix.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;EpiMix&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Running Enhancer mode...
Fetching probe annotation...
see ?sesameData and browseVignettes(&#x27;sesameData&#x27;) for documentation
loading from cache
require(&quot;GenomicRanges&quot;)
Found 457 samples in group.1 and 32 samples in group.2
Fetching enhancer CpGs from Roadmap Epigenomics...
Downloading chromatin states from the Roadmap Epigenomics...
trying URL &#x27;https://egg2.wustl.edu/roadmap/data/byFileType/chromhmmSegmentations/ChmmModels/core_K27ac/jointModel/final/E096_18_core_K27ac_hg38lift_mnemonics.bed.gz&#x27;
Content type &#x27;application/x-gzip&#x27; length 3835377 bytes (3.7 MB)
==================================================
downloaded 3.7 MB

	Identifed 65057 enhancer CpGs from the epigenome E096 
see ?sesameData and browseVignettes(&#x27;sesameData&#x27;) for documentation
loading from cache
Returning distal probes: 160862
Found 3 CpGs associated with distal enhancers in the methylation dataset

Starting Beta mixture modeling.
Running Beta mixture model on 3 probes and on 457 samples.
  |                                                                              |                                                                      |   0%
Found 3 differentially methylated CpGs
Modeling the gene expression for enhancers...
Searching for the 20 near genes
Identifying gene position for each probe
Looking for differentially methylated enhancers associated with gene expression
  |                                                                              |                                                                      |   0%Warning in getRandomGenes(target.probe = target.probe, gene.expression.data = gene.expression.data,  :
  There is not enough genes to generate 1000 permutations. Using 805 random genes instead.
  |                                                                              |=======================                                               |  33%Warning in getRandomGenes(target.probe = target.probe, gene.expression.data = gene.expression.data,  :
  There is not enough genes to generate 1000 permutations. Using 793 random genes instead.
  |                                                                              |===============================================                       |  67%Warning in getRandomGenes(target.probe = target.probe, gene.expression.data = gene.expression.data,  :
  There is not enough genes to generate 1000 permutations. Using 798 random genes instead.
  |                                                                              |======================================================================| 100%
Found 7 functional probe-gene pairs.
Saving the EpiMix results to the output directory...
Running lncRNA mode...
We recommend using the kallisto-sleuth pipline to process the RNA-seq data in order to detect more lncRNAs.
        Please see our publication for details: PMID: 31808800
Fetching probe annotation...
Found 457 samples in group.1 and 32 samples in group.2

Starting Beta mixture modeling.
Running Beta mixture model on 5 probes and on 457 samples.
  |                                                                              |                                                                      |   0%
Found 5 differentially methylated probes
Identifying functional CpG-gene pairs...
  |                                                                              |                                                                      |   0%  |                                                                              |===================================                                   |  50%  |                                                                              |======================================================================| 100%
Found 5 functional probe-gene pairs.
Saving the EpiMix results to the output directory...
Running miRNA mode...
Please be mindful that the gene expression data are expected to be data obtained from microRNA-seq.
Fetching probe annotation...
Found 457 samples in group.1 and 32 samples in group.2

Starting Beta mixture modeling.
Running Beta mixture model on 6 probes and on 457 samples.
  |                                                                              |                                                                      |   0%
Found 6 differentially methylated probes
Identifying functional CpG-gene pairs...
  |                                                                              |                                                                      |   0%  |                                                                              |=======================                                               |  33%  |                                                                              |===============================================                       |  67%  |                                                                              |======================================================================| 100%
Found 6 functional probe-gene pairs.
Saving the EpiMix results to the output directory...
Running Regular mode...
Fetching probe annotation...
see ?sesameData and browseVignettes(&#x27;sesameData&#x27;) for documentation
loading from cache
Found 474 samples with both methylation and gene expression data.
Modeling gene expression...
  |                                                                              |                                                                      |   0%  |                                                                              |=======                                                               |  10%  |                                                                              |==============                                                        |  20%  |                                                                              |=====================                                                 |  30%  |                                                                              |============================                                          |  40%  |                                                                              |===================================                                   |  50%  |                                                                              |==========================================                            |  60%  |                                                                              |=================================================                     |  70%  |                                                                              |========================================================              |  80%  |                                                                              |===============================================================       |  90%  |                                                                              |======================================================================| 100%
Found 5 transcriptionally predictive probes.
Found 457 samples in group.1 and 32 samples in group.2

Starting Beta mixture modeling.
Running Beta mixture model on 5 probes and on 457 samples.
  |                                                                              |                                                                      |   0%
Found 5 differentially methylated CpGs
Identifying functional CpG-gene pairs...
  |                                                                              |                                                                      |   0%  |                                                                              |==================                                                    |  25%  |                                                                              |===================================                                   |  50%  |                                                                              |====================================================                  |  75%  |                                                                              |======================================================================| 100%
Found 5 functional probe-gene pairs.
Saving the EpiMix results to the output directory...


RUNIT TEST PROTOCOL -- Sun Jun  4 16:29:23 2023 
*********************************************** 
Number of test functions: 4 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
EpiMix RUnit Tests - 4 test functions, 0 errors, 0 failures
Number of test functions: 4 
Number of errors: 0 
Number of failures: 0 
Warning message:
executing %dopar% sequentially: no parallel backend registered 
&gt; 
&gt; proc.time()
   user  system elapsed 
180.706   3.130 221.969 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">EpiMix.Rcheck/EpiMix-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>EpiMix</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EpiMix_PlotGene</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EpiMix_PlotModel</TD><TD style="text-align: right;">0.689</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.745</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EpiMix_PlotProbe</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EpiMix_PlotSurvival</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GEO_Download_DNAMethylation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GEO_Download_GeneExpression</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GetSurvivalProbe</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Preprocess_DNAMethylation</TD><TD style="text-align: right;">38.301</TD><TD style="text-align: right;"> 2.010</TD><TD style="text-align: right;">52.561</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Preprocess_GeneExpression</TD><TD style="text-align: right;">1.318</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.332</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TCGA_Download_DNAmethylation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TCGA_Download_GeneExpression</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TCGA_GetData</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TCGA_GetSampleInfo</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TCGA_Preprocess_DNAmethylation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TCGA_Preprocess_GeneExpression</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>find_miRNA_targets</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>functionEnrich</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getFunctionalGenes</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getRoadMapEnhancerProbes</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
