<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for MicrobiotaProcess on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><B>[<A href="..#M">M</A>]</B></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-06 11:00:36 -0000 (Tue, 06 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for MicrobiotaProcess on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the MicrobiotaProcess package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/MicrobiotaProcess.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1209</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">MicrobiotaProcess</A>&nbsp;1.13.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/MicrobiotaProcess">landing page</A>)</SPAN><BR>Shuangbin Xu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-05&nbsp;06:35:06&nbsp;-0000&nbsp;(Mon,&nbsp;05&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/MicrobiotaProcess</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">dc7add5</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-05-04&nbsp;02:02:50&nbsp;-0000&nbsp;(Thu,&nbsp;04&nbsp;May&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: MicrobiotaProcess</TD></TR>
<TR><TD><B>Version</B>: 1.13.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:MicrobiotaProcess.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings MicrobiotaProcess_1.13.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-06 02:03:06 -0000 (Tue, 06 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-06 02:11:52 -0000 (Tue, 06 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 525.7 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: MicrobiotaProcess.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:MicrobiotaProcess.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings MicrobiotaProcess_1.13.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/MicrobiotaProcess.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘MicrobiotaProcess/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘MicrobiotaProcess’ version ‘1.13.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘MicrobiotaProcess’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is  5.7Mb
  sub-directories of 1Mb or more:
    R         2.5Mb
    figures   1.9Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘MicrobiotaProcess-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: mp_diff_analysis
&gt; ### Title: Differential expression analysis for MPSE or tbl_mpse object
&gt; ### Aliases: mp_diff_analysis mp_diff_analysis,MPSE-method
&gt; ###   mp_diff_analysis,MPSE mp_diff_analysis,tbl_mpse-method
&gt; ###   mp_diff_analysis,tbl_mpse mp_diff_analysis,grouped_df_mpse-method
&gt; ###   mp_diff_analysis,grouped_df_mpse
&gt; 
&gt; ### ** Examples
&gt; 
&gt; data(mouse.time.mpse)
&gt; mouse.time.mpse %&lt;&gt;%
+   mp_rrarefy() 
&gt; mouse.time.mpse
# A MPSE-tibble (MPSE object) abstraction: 4,142 × 12
# OTU=218 | Samples=19 | Assays=Abundance, RareAbundance | Taxonomy=Kingdom,
#   Phylum, Class, Order, Family, Genus, Species
   OTU    Sample Abundance RareAbundance time  Kingdom Phylum Class Order Family
   &lt;chr&gt;  &lt;chr&gt;      &lt;int&gt;         &lt;int&gt; &lt;chr&gt; &lt;chr&gt;   &lt;chr&gt;  &lt;chr&gt; &lt;chr&gt; &lt;chr&gt; 
 1 OTU_1  F3D0         579           214 Early k__Bac… p__Ba… c__B… o__B… f__Mu…
 2 OTU_2  F3D0         345           116 Early k__Bac… p__Ba… c__B… o__B… f__Mu…
 3 OTU_3  F3D0         449           179 Early k__Bac… p__Ba… c__B… o__B… f__Mu…
 4 OTU_4  F3D0         430           167 Early k__Bac… p__Ba… c__B… o__B… f__Mu…
 5 OTU_5  F3D0         154            54 Early k__Bac… p__Ba… c__B… o__B… f__Ba…
 6 OTU_6  F3D0         470           174 Early k__Bac… p__Ba… c__B… o__B… f__Mu…
 7 OTU_7  F3D0         282           115 Early k__Bac… p__Ba… c__B… o__B… f__Mu…
 8 OTU_8  F3D0         184            74 Early k__Bac… p__Ba… c__B… o__B… f__Ri…
 9 OTU_9  F3D0          45            16 Early k__Bac… p__Ba… c__B… o__B… f__Mu…
10 OTU_10 F3D0         158            59 Early k__Bac… p__Ba… c__B… o__B… f__Mu…
# ℹ 4,132 more rows
# ℹ 2 more variables: Genus &lt;chr&gt;, Species &lt;chr&gt;
&gt; mouse.time.mpse %&lt;&gt;%
+   mp_diff_analysis(.abundance=RareAbundance, 
+                    .group=time, 
+                    first.test.alpha=0.01,
+                    action=&quot;add&quot;) 
&gt; library(ggplot2)
&gt; p &lt;- mouse.time.mpse %&gt;% mp_plot_diff_res()
Warning: Duplicated aesthetics after name standardisation: fill
Error in new_stat$default_aes[[new_aes]] : no such index at level 1
Calls: %&gt;% -&gt; mp_plot_diff_res -&gt; mp_plot_diff_res
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘MicrobiotaProcess.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/MicrobiotaProcess.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">MicrobiotaProcess.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL MicrobiotaProcess
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘MicrobiotaProcess’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
in method for ‘taxonomy&lt;-’ with signature ‘x=&quot;MPSE&quot;,value=&quot;taxonomyTable&quot;’: no definition for class “taxonomyTable”
Creating a generic function for ‘rownames&lt;-’ from package ‘base’ in package ‘MicrobiotaProcess’
in method for ‘get_alltaxadf’ with signature ‘&quot;phyloseq&quot;’: no definition for class “phyloseq”
in method for ‘get_taxadf’ with signature ‘&quot;phyloseq&quot;’: no definition for class “phyloseq”
in method for ‘get_alphaindex’ with signature ‘&quot;phyloseq&quot;’: no definition for class “phyloseq”
in method for ‘get_NRI_NTI’ with signature ‘&quot;phyloseq&quot;’: no definition for class “phyloseq”
in method for ‘get_rarecurve’ with signature ‘&quot;phyloseq&quot;’: no definition for class “phyloseq”
in method for ‘get_upset’ with signature ‘&quot;phyloseq&quot;’: no definition for class “phyloseq”
in method for ‘get_vennlist’ with signature ‘&quot;phyloseq&quot;’: no definition for class “phyloseq”
in method for ‘drop_taxa’ with signature ‘&quot;phyloseq&quot;’: no definition for class “phyloseq”
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (MicrobiotaProcess)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">MicrobiotaProcess.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;testthat&quot;)
&gt; library(&quot;MicrobiotaProcess&quot;)
MicrobiotaProcess v1.13.1 For help:
https://github.com/YuLab-SMU/MicrobiotaProcess/issues

If you use MicrobiotaProcess in published research, please cite the
paper:

Shuangbin Xu, Li Zhan, Wenli Tang, Qianwen Wang, Zehan Dai, Lang Zhou,
Tingze Feng, Meijun Chen, Tianzhi Wu, Erqiang Hu, Guangchuang Yu.
MicrobiotaProcess: A comprehensive R package for deep mining
microbiome. The Innovation. 2023, 4(2):100388. doi:
10.1016/j.xinn.2023.100388

Export the citation to BibTex by citation(&#x27;MicrobiotaProcess&#x27;)

This message can be suppressed by:
suppressPackageStartupMessages(library(MicrobiotaProcess))

Attaching package: &#x27;MicrobiotaProcess&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

&gt; test_check(&quot;MicrobiotaProcess&quot;)
[ FAIL 0 | WARN 9 | SKIP 0 | PASS 13 ]

[ FAIL 0 | WARN 9 | SKIP 0 | PASS 13 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 17.391   0.679  18.068 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">MicrobiotaProcess.Rcheck/MicrobiotaProcess-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ImportDada2</TD><TD style="text-align: right;">2.718</TD><TD style="text-align: right;">0.164</TD><TD style="text-align: right;">2.891</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ImportQiime2</TD><TD style="text-align: right;">4.809</TD><TD style="text-align: right;">0.111</TD><TD style="text-align: right;">4.963</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MPSE</TD><TD style="text-align: right;">0.139</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.143</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>as.treedata</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>build_tree</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convert_to_treedata</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-hmp_aerobiosis_small</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-kostic2012crc</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-test_otu_data</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>diff_analysis</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dr_extract</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>drop_taxa</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generalizedFC</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_alltaxadf</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_alphaindex</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_clust</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_coord</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_count</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_dist</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_mean_median</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_pca</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_pcoa</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_pvalue</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_rarecurve</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_sampledflist</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_taxadf</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_upset</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_varct</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_vennlist</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggbartax</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggbox</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggclust</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggdiffbox</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggdiffclade</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggdifftaxbar</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggeffectsize</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggordpoint</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ggrarecurve</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_adonis-methods</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.135</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_aggregate-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_aggregate_clade-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_anosim-methods</TD><TD style="text-align: right;">1.624</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.644</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_balance_clade-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_abundance-methods</TD><TD style="text-align: right;">12.932</TD><TD style="text-align: right;"> 0.202</TD><TD style="text-align: right;">13.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_alpha-methods</TD><TD style="text-align: right;">1.555</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">1.593</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_cca-methods</TD><TD style="text-align: right;">1.221</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">1.275</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_clust-methods</TD><TD style="text-align: right;">0.701</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.710</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_dist-methods</TD><TD style="text-align: right;">2.921</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">2.951</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_divergence-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_nmds-methods</TD><TD style="text-align: right;">0.328</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.332</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_pca-methods</TD><TD style="text-align: right;">2.130</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.150</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_pcoa-methods</TD><TD style="text-align: right;">0.799</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.804</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_pd_metric-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_rarecurve-methods</TD><TD style="text-align: right;">8.561</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">8.668</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_rda-methods</TD><TD style="text-align: right;">1.887</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">1.954</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_upset-methods</TD><TD style="text-align: right;">1.447</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.456</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_cal_venn-methods</TD><TD style="text-align: right;">2.004</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">2.083</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mp_decostand-methods</TD><TD style="text-align: right;">0.462</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.463</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
