<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for diffHic on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><B>[<A href="..#D">D</A>]</B></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-25 05:45:53 -0000 (Thu, 25 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4223</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for diffHic on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the diffHic package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/diffHic.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>547</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">diffHic</A>&nbsp;1.33.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/diffHic">landing page</A>)</SPAN><BR>Aaron Lun<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-24&nbsp;15:28:50&nbsp;-0000&nbsp;(Wed,&nbsp;24&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/diffHic</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">8da9340</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:39:29&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: diffHic</TD></TR>
<TR><TD><B>Version</B>: 1.33.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:diffHic.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings diffHic_1.33.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-25 04:33:03 -0000 (Thu, 25 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-25 04:33:23 -0000 (Thu, 25 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 20.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: diffHic.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:diffHic.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings diffHic_1.33.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/diffHic.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘diffHic/DESCRIPTION’ ... OK
* this is package ‘diffHic’ version ‘1.33.0’
* checking package namespace information ... OK
* checking package dependencies ...Warning: unable to access index for repository https://CRAN.R-project.org/src/contrib:
  download from &#x27;https://CRAN.R-project.org/src/contrib/PACKAGES&#x27; failed
Warning: unable to access index for repository https://bioconductor.org/packages/3.17/bioc/src/contrib:
  download from &#x27;https://bioconductor.org/packages/3.17/bioc/src/contrib/PACKAGES&#x27; failed
Warning: unable to access index for repository https://bioconductor.org/packages/3.17/data/annotation/src/contrib:
  download from &#x27;https://bioconductor.org/packages/3.17/data/annotation/src/contrib/PACKAGES&#x27; failed
Warning: unable to access index for repository https://bioconductor.org/packages/3.17/data/experiment/src/contrib:
  download from &#x27;https://bioconductor.org/packages/3.17/data/experiment/src/contrib/PACKAGES&#x27; failed
 OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘diffHic’ can be installed ... OK
* used C++ compiler: ‘g++ (GCC) 10.3.1’
* checking C++ specification ... NOTE
  Specified C++11: please drop specification unless essential
* checking installed package size ... NOTE
  installed size is 16.5Mb
  sub-directories of 1Mb or more:
    doc    1.3Mb
    libs  13.9Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... NOTE
Problems with news in ‘inst/NEWS.Rd’:
  Error writing to connection:  No space left on device
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ...Warning in for (i in seq_along(specs)) { :
  closing unused connection 4 ()
 OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... OK
* checking installed files from ‘inst/doc’ ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... NONE
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ... OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘diffHic.Rnw’... failed to complete the test
 ERROR
Errors in running code in vignettes:
when running code in ‘diffHic.Rnw’
  ...
&gt; proc.time()
   user  system elapsed 
  0.157   0.040   0.193 

... incomplete output.  Crash?

* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... ERROR
Rd conversion errors:
Converting parsed Rd&#x27;s to LaTeX Warning in close.connection(con) :
  Problem closing connection:  No space left on device
Warning in close.connection(con) :
  Problem closing connection:  No space left on device
Warning in close.connection(con) :
  Problem closing connection:  No space left on device
Warning in close.connection(con) :
  Problem closing connection:  No space left on device
Warning in close.connection(con) :
  Problem closing connection:  No space left on device
Error in writeLines(x, con, useBytes = TRUE, ...) : 
  Error writing to connection:  No space left on device
* DONE

Status: 2 ERRORs, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/diffHic.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">diffHic.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL diffHic
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘diffHic’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (GCC) 10.3.1’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c binner.cpp -o binner.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c check_input.cpp -o check_input.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c cluster_2d.cpp -o cluster_2d.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c count_background.cpp -o count_background.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c count_connect.cpp -o count_connect.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c count_patch.cpp -o count_patch.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c directionality.cpp -o directionality.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c init.cpp -o init.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c iterative_correction.cpp -o iterative_correction.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c neighbors.cpp -o neighbors.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c pair_stats.cpp -o pair_stats.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c quadrant_bg.cpp -o quadrant_bg.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c report_hic_pairs.cpp -o report_hic_pairs.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c trended_filter.cpp -o trended_filter.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c utils.cpp -o utils.o
g++ -std=gnu++11 -shared -L/home/biocbuild/R/R-4.3.0/lib -L/usr/local/lib -o diffHic.so binner.o check_input.o cluster_2d.o count_background.o count_connect.o count_patch.o directionality.o init.o iterative_correction.o neighbors.o pair_stats.o quadrant_bg.o report_hic_pairs.o trended_filter.o utils.o /home/biocbuild/R/R-4.3.0/site-library/Rhtslib/usrlib/libhts.a -lcurl -L/home/biocbuild/R/R-4.3.0/lib -lR
installing to /home/biocbuild/R/R-4.3.0/site-library/00LOCK-diffHic/00new/diffHic/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (diffHic)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">diffHic.Rcheck/tests/test-basic.Rout.save</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 3.5.0 Patched (2018-04-30 r74679) -- &quot;Joy in Playing&quot;
Copyright (C) 2018 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; # These are just placeholders for the real things in inst/tests.
&gt; 
&gt; suppressWarnings(suppressPackageStartupMessages(require(diffHic)))
&gt; 
&gt; hic.file &lt;- system.file(&quot;exdata&quot;, &quot;hic_sort.bam&quot;, package=&quot;diffHic&quot;)
&gt; cuts &lt;- readRDS(system.file(&quot;exdata&quot;, &quot;cuts.rds&quot;, package=&quot;diffHic&quot;))
&gt; param &lt;- pairParam(fragments=cuts)
&gt; 
&gt; # Setting up the parameters
&gt; fout &lt;- &quot;output.h5&quot;
&gt; preparePairs(hic.file, param, file=fout)
$pairs
   total   marked filtered   mapped 
      32        7        3       22 

$same.id
   dangling self.circle 
          4           1 

$singles
[1] 2

$chimeras
  total  mapped   multi invalid 
     12       8       7       5 

&gt; head(getPairData(fout, param))
  length orientation insert
1     40           1     40
2     80           1     80
3     60           2     54
4     40           3    160
5     20           1     66
6     80           0    114
&gt; 
&gt; loadChromos(fout)
  anchor1 anchor2
1    chrA    chrA
2    chrB    chrA
3    chrB    chrB
&gt; head(loadData(fout, &quot;chrA&quot;, &quot;chrA&quot;))
  anchor1.id anchor2.id anchor1.pos anchor2.pos anchor1.len anchor2.len
1          2          1          49          14          -5          10
2          2          1          79           9         -10          10
3          2          1          65          21          10         -10
4          4          1         156           6         -10         -10
5          4          2         141          80          -5          10
6          4          2         154          50          10          10
&gt; head(loadData(fout, &quot;chrA&quot;, &quot;chrB&quot;))
  anchor1.id anchor2.id anchor1.pos anchor2.pos anchor1.len anchor2.len
1          5          2          19          44          10         -10
2          5          3          14         105          10         -10
3          6          1          65          19           5          10
4          6          2          24          90          -5           5
5          6          3          24         100          -5         -10
6          6          3          24          95          -5         -10
Warning message:
In value[[3L]](cond) : anchor definitions are reversed
&gt; 
&gt; # Loading the counts.
&gt; data &lt;- squareCounts(fout, param, width=50, filter=1)
&gt; data
class: InteractionSet 
dim: 10 1 
metadata(2): param width
assays(1): counts
rownames: NULL
rowData names(0):
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 6
&gt; 
&gt; margins &lt;- marginCounts(fout, param, width=50)
&gt; margins
class: RangedSummarizedExperiment 
dim: 6 1 
metadata(1): param
assays(1): counts
rownames: NULL
rowData names(1): nfrags
colnames: NULL
colData names(1): totals
&gt; totalCounts(fout, param)
[1] 17
&gt; 
&gt; regions &lt;- GRanges(&quot;chrA&quot;, IRanges(c(1, 100, 150), c(20, 140, 160)))
&gt; connectCounts(fout, param, regions=regions, filter=1L)
class: InteractionSet 
dim: 2 1 
metadata(1): param
assays(1): counts
rownames: NULL
rowData names(0):
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 3
&gt; 
&gt; # Checking some values.
&gt; head(getArea(data))
[1] 2208 2304 2208 2208 3312 3174
&gt; head(pairdist(data))
[1]  47 140  93  47  NA  NA
&gt; 
&gt; anchors(data, type=&quot;first&quot;)
GRanges object with 10 ranges and 1 metadata column:
       seqnames    ranges strand |    nfrags
          &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
   [1]     chrA     49-94      * |         1
   [2]     chrA   141-188      * |         1
   [3]     chrA   141-188      * |         1
   [4]     chrA   141-188      * |         1
   [5]     chrB      1-69      * |         2
   [6]     chrB      1-69      * |         2
   [7]     chrB      1-69      * |         2
   [8]     chrB      1-69      * |         2
   [9]     chrB     70-92      * |         1
  [10]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; anchors(data, type=&quot;second&quot;)
GRanges object with 10 ranges and 1 metadata column:
       seqnames    ranges strand |    nfrags
          &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
   [1]     chrA      1-48      * |         1
   [2]     chrA      1-48      * |         1
   [3]     chrA     49-94      * |         1
   [4]     chrA    95-140      * |         1
   [5]     chrA      1-48      * |         1
   [6]     chrA     49-94      * |         1
   [7]     chrA    95-140      * |         1
   [8]     chrB      1-69      * |         2
   [9]     chrB      1-69      * |         2
  [10]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; assay(data)
      [,1]
 [1,]    3
 [2,]    1
 [3,]    2
 [4,]    1
 [5,]    1
 [6,]    2
 [7,]    3
 [8,]    1
 [9,]    2
[10,]    1
&gt; regions(data)
GRanges object with 6 ranges and 1 metadata column:
      seqnames    ranges strand |    nfrags
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
  [1]     chrA      1-48      * |         1
  [2]     chrA     49-94      * |         1
  [3]     chrA    95-140      * |         1
  [4]     chrA   141-188      * |         1
  [5]     chrB      1-69      * |         2
  [6]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; 
&gt; data$totals
[1] 17
&gt; colData(data)
DataFrame with 1 row and 1 column
     totals
  &lt;integer&gt;
1        17
&gt; metadata(data)
$param
Genome contains 7 restriction fragments across 2 chromosomes
No discard regions are specified
No limits on chromosomes for read extraction
No cap on the read pairs per pair of restriction fragments

$width
[1] 50

&gt; 
&gt; asDGEList(data)
An object of class &quot;DGEList&quot;
$counts
   Sample1
1        3
2        1
3        2
4        1
5        1
6        2
7        3
8        1
9        2
10       1

$samples
        group lib.size norm.factors
Sample1     1       17            1

&gt; asDGEList(data, lib.size=20)$samples
        group lib.size norm.factors
Sample1     1       20            1
&gt; asDGEList(data, norm.factors=2, group=&quot;a&quot;)$samples
        group lib.size norm.factors
Sample1     a       17            2
&gt; 
&gt; # Playing around with some bin counts.
&gt; stuff &lt;- correctedContact(data)
&gt; head(stuff$truth)
[1] 1.517032e+04 6.123523e-01 6.123523e-01 8.162749e-01 2.041633e-01
[6] 2.041633e-01
&gt; 
&gt; data.large &lt;- squareCounts(fout, param, width=100, filter=1)
&gt; boxed &lt;- boxPairs(larger=data.large, smaller=data)
&gt; head(boxed$indices$larger)
[1] 1 2 3 4 5 6
&gt; head(boxed$indices$smaller)
[1] 1 2 2 3 4 4
&gt; 
&gt; head(enrichedPairs(data))
class: InteractionSet 
dim: 6 1 
metadata(2): param width
assays(5): counts quadrant vertical horizontal surrounding
rownames: NULL
rowData names(4): N.quadrant N.vertical N.horizontal N.surrounding
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 6
&gt; head(clusterPairs(data, tol=10)$indices[[1]])
[1] 1 2 2 2 3 3
&gt; 
&gt; # End.
&gt; 
&gt; unlink(fout)
&gt; 
&gt; proc.time()
   user  system elapsed 
  5.571   0.168   5.757 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
</BODY>
</HTML>
