<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for oligo on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><B>[<A href="..#O">O</A>]</B></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-06 11:00:37 -0000 (Tue, 06 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for oligo on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the oligo package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/oligo.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1388</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">oligo</A>&nbsp;1.65.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/oligo">landing page</A>)</SPAN><BR>Benilton Carvalho<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-05&nbsp;06:35:06&nbsp;-0000&nbsp;(Mon,&nbsp;05&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/oligo</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">261cd56</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;13:42:11&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: oligo</TD></TR>
<TR><TD><B>Version</B>: 1.65.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:oligo.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings oligo_1.65.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-06 03:21:29 -0000 (Tue, 06 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-06 03:29:16 -0000 (Tue, 06 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 466.7 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: oligo.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 6</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:oligo.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings oligo_1.65.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/oligo.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘oligo/DESCRIPTION’ ... OK
* this is package ‘oligo’ version ‘1.65.0’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘doMPI’
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘oligo’ can be installed ... WARNING
Found the following significant warnings:
  ParserGzXYS.c:18:17: warning: passing argument 1 of ‘gzeof’ from incompatible pointer type [-Wincompatible-pointer-types]
  ParserGzXYS.c:19:12: warning: passing argument 1 of ‘gzgets’ from incompatible pointer type [-Wincompatible-pointer-types]
  ParserGzXYS.c:46:6: warning: assignment to ‘struct gzFile_s **’ from incompatible pointer type ‘gzFile’ {aka ‘struct gzFile_s *’} [-Wincompatible-pointer-types]
  ParserGzXYS.c:50:10: warning: passing argument 1 of ‘gzgets’ from incompatible pointer type [-Wincompatible-pointer-types]
  ParserGzXYS.c:51:11: warning: passing argument 1 of ‘gzclose’ from incompatible pointer type [-Wincompatible-pointer-types]
  ParserGzXYS.c:134:6: warning: assignment to ‘struct gzFile_s **’ from incompatible pointer type ‘gzFile’ {aka ‘struct gzFile_s *’} [-Wincompatible-pointer-types]
  ParserGzXYS.c:138:11: warning: passing argument 1 of ‘gzclose’ from incompatible pointer type [-Wincompatible-pointer-types]
See ‘/home/biocbuild/bbs-3.18-bioc/meat/oligo.Rcheck/00install.out’ for details.
* used C compiler: ‘gcc (GCC) 10.3.1’
* checking installed package size ... NOTE
  installed size is 17.1Mb
  sub-directories of 1Mb or more:
    scripts  15.7Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘oligo’ for: ‘show’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... NOTE
Unexported object imported by a &#x27;:::&#x27; call: ‘Biobase:::annotatedDataFrameFromMatrix’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... NOTE
Foreign function calls to a different package:
  .Call(&quot;ReadHeader&quot;, ..., PACKAGE = &quot;affyio&quot;)
  .Call(&quot;read_abatch&quot;, ..., PACKAGE = &quot;affyio&quot;)
See chapter ‘System and foreign language interfaces’ in the ‘Writing R
Extensions’ manual.
* checking R code for possible problems ... NOTE
image,FeatureSet: warning in matrix(NA, nr = geom[1], nc = geom[2]):
  partial argument match of &#x27;nr&#x27; to &#x27;nrow&#x27;
image,FeatureSet: warning in matrix(NA, nr = geom[1], nc = geom[2]):
  partial argument match of &#x27;nc&#x27; to &#x27;ncol&#x27;
cloneFS: no visible global function definition for ‘clone’
rmaBgCorrectLDSmaster: no visible global function definition for
  ‘clone’
backgroundCorrect,matrix: no visible binding for global variable
  ‘intensities’
normalize,ff_matrix: no visible global function definition for ‘clone’
normalizeToTarget,ff_matrix: no visible global function definition for
  ‘clone’
pmindex,GenericPDInfo: no visible binding for global variable
  ‘man_fsetid’
Undefined global functions or variables:
  clone intensities man_fsetid
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic &#x27;mm&lt;-&#x27; and siglist &#x27;TilingFeatureSet,missing,missing,array&#x27;
  generic &#x27;pm&lt;-&#x27; and siglist &#x27;TilingFeatureSet,missing,missing,array&#x27;
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object &#x27;boxplot&#x27;
  ‘target’

Undocumented arguments in documentation object &#x27;hist&#x27;
  ‘target’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in shell scripts ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... WARNING
  apparently using $(BLAS_LIBS) without following $(FLIBS) in ‘src/Makevars’
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking compiled code ... WARNING
Note: information on .o files is not available
File ‘/home/biocbuild/R/R-4.3.0/site-library/oligo/libs/oligo.so’:
  Found ‘sprintf’, possibly from ‘sprintf’ (C)

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor use Fortran I/O
nor system RNGs nor [v]sprintf. The detected symbols are linked into
the code but might come from libraries and not actually be called.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
* checking files in ‘vignettes’ ... WARNING
Files in the &#x27;vignettes&#x27; directory newer than all files in &#x27;inst/doc&#x27;:
  ‘Makefile’
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                  user system elapsed
preprocessTools 20.175  0.304  22.131
getProbeInfo    12.218  1.869  14.451
read.celfiles    4.961  0.411   5.756
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘doRUnit.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘oug.Rnw’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 6 WARNINGs, 6 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/oligo.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">oligo.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL oligo
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘oligo’ ...
** using staged installation
** libs
using C compiler: ‘gcc (GCC) 10.3.1’
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c DABG.c -o DABG.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c ParserGzXYS.c -o ParserGzXYS.o
ParserGzXYS.c: In function ‘gzcountLines’:
ParserGzXYS.c:18:17: warning: passing argument 1 of ‘gzeof’ from incompatible pointer type [-Wincompatible-pointer-types]
   18 |   while (!gzeof(file)){
      |                 ^~~~
      |                 |
      |                 struct gzFile_s **
In file included from /usr/include/zlib.h:34,
                 from ParserGzXYS.c:4:
/usr/include/zlib.h:1594:27: note: expected ‘gzFile’ {aka ‘struct gzFile_s *’} but argument is of type ‘struct gzFile_s **’
 1594 | ZEXTERN int ZEXPORT gzeof OF((gzFile file));
      |                           ^~
ParserGzXYS.c:19:12: warning: passing argument 1 of ‘gzgets’ from incompatible pointer type [-Wincompatible-pointer-types]
   19 |     gzgets(file, buffer, 1000);
      |            ^~~~
      |            |
      |            struct gzFile_s **
In file included from /usr/include/zlib.h:34,
                 from ParserGzXYS.c:4:
/usr/include/zlib.h:1491:31: note: expected ‘gzFile’ {aka ‘struct gzFile_s *’} but argument is of type ‘struct gzFile_s **’
 1491 | ZEXTERN char * ZEXPORT gzgets OF((gzFile file, char *buf, int len));
      |                               ^~
ParserGzXYS.c: In function ‘gzxys_header_field’:
ParserGzXYS.c:46:6: warning: assignment to ‘struct gzFile_s **’ from incompatible pointer type ‘gzFile’ {aka ‘struct gzFile_s *’} [-Wincompatible-pointer-types]
   46 |   fp = gzopen(currentFile, &quot;rb&quot;);
      |      ^
ParserGzXYS.c:50:10: warning: passing argument 1 of ‘gzgets’ from incompatible pointer type [-Wincompatible-pointer-types]
   50 |   gzgets(fp, buffer, LINEMAX);
      |          ^~
      |          |
      |          struct gzFile_s **
In file included from /usr/include/zlib.h:34,
                 from ParserGzXYS.c:4:
/usr/include/zlib.h:1491:31: note: expected ‘gzFile’ {aka ‘struct gzFile_s *’} but argument is of type ‘struct gzFile_s **’
 1491 | ZEXTERN char * ZEXPORT gzgets OF((gzFile file, char *buf, int len));
      |                               ^~
ParserGzXYS.c:51:11: warning: passing argument 1 of ‘gzclose’ from incompatible pointer type [-Wincompatible-pointer-types]
   51 |   gzclose(fp);
      |           ^~
      |           |
      |           struct gzFile_s **
In file included from /usr/include/zlib.h:34,
                 from ParserGzXYS.c:4:
/usr/include/zlib.h:1630:32: note: expected ‘gzFile’ {aka ‘struct gzFile_s *’} but argument is of type ‘struct gzFile_s **’
 1630 | ZEXTERN int ZEXPORT    gzclose OF((gzFile file));
      |                                ^~
ParserGzXYS.c: In function ‘R_read_gzxys_files’:
ParserGzXYS.c:134:6: warning: assignment to ‘struct gzFile_s **’ from incompatible pointer type ‘gzFile’ {aka ‘struct gzFile_s *’} [-Wincompatible-pointer-types]
  134 |   fp = gzopen(CHAR(STRING_ELT(filenames, 0)), &quot;rb&quot;);
      |      ^
ParserGzXYS.c:138:11: warning: passing argument 1 of ‘gzclose’ from incompatible pointer type [-Wincompatible-pointer-types]
  138 |   gzclose(fp);
      |           ^~
      |           |
      |           struct gzFile_s **
In file included from /usr/include/zlib.h:34,
                 from ParserGzXYS.c:4:
/usr/include/zlib.h:1630:32: note: expected ‘gzFile’ {aka ‘struct gzFile_s *’} but argument is of type ‘struct gzFile_s **’
 1630 | ZEXTERN int ZEXPORT    gzclose OF((gzFile file));
      |                                ^~
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c ParserXYS.c -o ParserXYS.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c baseProfile.c -o baseProfile.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c basecontent.c -o basecontent.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c chipbackground.c -o chipbackground.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c mas5calls.c -o mas5calls.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c rma2.c -o rma2.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c rma_common.c -o rma_common.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c trimmed.c -o trimmed.o
gcc -shared -L/home/biocbuild/R/R-4.3.0/lib -L/usr/local/lib -o oligo.so DABG.o ParserGzXYS.o ParserXYS.o baseProfile.o basecontent.o chipbackground.o mas5calls.o rma2.o rma_common.o trimmed.o -lz -lgfortran -lm -llapack -lblas -L/home/biocbuild/R/R-4.3.0/lib -lR
installing to /home/biocbuild/R/R-4.3.0/site-library/00LOCK-oligo/00new/oligo/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Note: in method for ‘pm&lt;-’ with signature
‘object=&quot;TilingFeatureSet&quot;,subset=&quot;ANY&quot;,target=&quot;ANY&quot;,value=&quot;array&quot;’:
expanding the signature to include omitted arguments in definition:
subset = &quot;missing&quot;, target = &quot;missing&quot;
Note: in method for ‘mm&lt;-’ with signature
‘object=&quot;TilingFeatureSet&quot;,subset=&quot;ANY&quot;,target=&quot;ANY&quot;,value=&quot;array&quot;’:
expanding the signature to include omitted arguments in definition:
subset = &quot;missing&quot;, target = &quot;missing&quot;
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (oligo)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">oligo.Rcheck/tests/doRUnit.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; ## from xmapcore package
&gt; if( require( &quot;RUnit&quot;, quietly=TRUE ) ) {
+ 	pkg &lt;- &quot;oligo&quot;
+         path &lt;- ifelse(Sys.getenv( &quot;RCMDCHECK&quot; ) == &quot;FALSE&quot;,
+                        file.path( getwd(), &quot;..&quot;, &quot;inst&quot;, &quot;unitTests&quot; ),
+                        system.file( package=pkg, &quot;unitTests&quot; ))
+         
+ 	cat( &quot;\nRunning unit tests\n&quot; )
+ 	print( list( pkg=pkg, getwd=getwd(), pathToUnitTests=path ) )
+ 	library( package=pkg, character.only=TRUE )
+ 
+ 	##Fail on warnings
+ 	##options( warn=2 )
+ 	options(warn=0)
+ 
+ 	## Get the pattern (if there is one?)
+ 	patt &lt;- Sys.getenv( &quot;RUNITFILEPATTERN&quot; )
+         testSuite &lt;- defineTestSuite(name=paste( pkg, &quot;unit testing&quot; ),
+                                      dirs=path,
+                                      testFileRegexp=paste( &quot;^test.+&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ))
+ 	tests &lt;- runTestSuite( testSuite )
+ 
+ 	pathReport &lt;- file.path( path, &quot;report&quot; )
+ 
+ 	cat( &quot;------------------- UNIT TEST SUMMARY ---------------------\n\n&quot; )
+ 	printTextProtocol( tests, showDetails=FALSE )
+ 	printTextProtocol( tests, showDetails=FALSE, fileName=paste( pathReport, &quot;Summary.txt&quot;, sep=&quot;&quot; ) )
+ 	printTextProtocol( tests, showDetails=TRUE,  fileName=paste( pathReport, &quot;.txt&quot;, sep=&quot;&quot; ) )
+ 
+ 	printHTMLProtocol( tests, fileName=paste( pathReport, &quot;.html&quot;, sep=&quot;&quot; ) )
+ 
+ 	tmp &lt;- getErrors( tests )
+ 	if( tmp$nFail &gt; 0 | tmp$nErr &gt; 0 ){
+ 		stop( paste( &quot;\n\nunit testing failed (#test failures: &quot;, tmp$nFail, &quot;, #R errors: &quot;,  tmp$nErr, &quot;)\n\n&quot;, sep=&quot;&quot;))
+ 	}
+ } else {
+ 	warning( &quot;cannot run unit tests -- package RUnit is not available&quot; )
+ }

Running unit tests
$pkg
[1] &quot;oligo&quot;

$getwd
[1] &quot;/home/biocbuild/bbs-3.18-bioc/meat/oligo.Rcheck/tests&quot;

$pathToUnitTests
[1] &quot;/tmp/RtmpLPvGWa/RLIBS_7e9a27a2ea49d/oligo/unitTests&quot;

Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: oligoClasses
Welcome to oligoClasses version 1.63.0
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: Biostrings
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: XVector
Loading required package: GenomeInfoDb

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

================================================================================
Welcome to oligo version 1.65.0
================================================================================


Executing test function test_rma  ... Getting sample dataset
Loading required package: pd.hg18.60mer.expr
Loading required package: RSQLite
Loading required package: DBI
Platform design info loaded.
Checking designs for each XYS file... Done.
Allocating memory... Done.
Reading /tmp/RtmpLPvGWa/RLIBS_7e9a27a2ea49d/maqcExpression4plex/extdata/9868701_532.xys.
Reading /tmp/RtmpLPvGWa/RLIBS_7e9a27a2ea49d/maqcExpression4plex/extdata/9868901_532.xys.
Reading /tmp/RtmpLPvGWa/RLIBS_7e9a27a2ea49d/maqcExpression4plex/extdata/9869001_532.xys.
Reading /tmp/RtmpLPvGWa/RLIBS_7e9a27a2ea49d/maqcExpression4plex/extdata/9870301_532.xys.
Reading /tmp/RtmpLPvGWa/RLIBS_7e9a27a2ea49d/maqcExpression4plex/extdata/9870401_532.xys.
Reading /tmp/RtmpLPvGWa/RLIBS_7e9a27a2ea49d/maqcExpression4plex/extdata/9870601_532.xys.
Running RMA
Background correcting
Normalizing
Calculating Expression
Getting reference results
 done successfully.



Executing test function test_selector  ... Loading reference data
Loading sample dataset: Exon
Getting probe info: core
Loading required package: pd.huex.1.0.st.v2
Getting probe info: probeset
Getting probe info: antigenomic backgroung probes
Loading sample dataset: Gene
Getting probe info: core
Loading required package: pd.hugene.1.0.st.v1
Getting probe info: probeset
Getting probe info: antigenomic backgroung probes
 done successfully.

------------------- UNIT TEST SUMMARY ---------------------

RUNIT TEST PROTOCOL -- Tue Jun  6 03:29:07 2023 
*********************************************** 
Number of test functions: 2 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
oligo unit testing - 2 test functions, 0 errors, 0 failures
Warning messages:
1: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
2: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
3: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
4: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
&gt; 
&gt; proc.time()
   user  system elapsed 
105.802   6.223 128.091 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">oligo.Rcheck/oligo-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>Index-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IntensityMatrix-methods</TD><TD style="text-align: right;">0.866</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">1.514</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MAplot-methods</TD><TD style="text-align: right;">2.370</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">2.870</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>basecontent</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>basicPLM</TD><TD style="text-align: right;">1.643</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.957</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>basicRMA</TD><TD style="text-align: right;">1.636</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">2.256</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>colors</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coordinates</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fitProbeLevelModel</TD><TD style="text-align: right;">4.233</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">4.914</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getProbeInfo</TD><TD style="text-align: right;">12.218</TD><TD style="text-align: right;"> 1.869</TD><TD style="text-align: right;">14.451</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>image</TD><TD style="text-align: right;">1.089</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.102</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>justSNPRMA</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list.xysfiles</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>oligoPLM-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>paCalls</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>preprocessTools</TD><TD style="text-align: right;">20.175</TD><TD style="text-align: right;"> 0.304</TD><TD style="text-align: right;">22.131</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read.celfiles</TD><TD style="text-align: right;">4.961</TD><TD style="text-align: right;">0.411</TD><TD style="text-align: right;">5.756</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read.xysfiles</TD><TD style="text-align: right;">0.547</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.671</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rma-methods</TD><TD style="text-align: right;">1.910</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">2.093</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sequenceDesignMatrix</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
