<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for GBScleanR on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><B>[<A href="..#G">G</A>]</B></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-06 11:00:32 -0000 (Tue, 06 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for GBScleanR on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GBScleanR package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GBScleanR.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>748</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GBScleanR</A>&nbsp;1.5.3</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/GBScleanR">landing page</A>)</SPAN><BR>Tomoyuki Furuta<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-05&nbsp;06:35:06&nbsp;-0000&nbsp;(Mon,&nbsp;05&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GBScleanR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">1c97903</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-05-11&nbsp;08:40:52&nbsp;-0000&nbsp;(Thu,&nbsp;11&nbsp;May&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GBScleanR</TD></TR>
<TR><TD><B>Version</B>: 1.5.3</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:GBScleanR.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings GBScleanR_1.5.3.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-05 22:43:19 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-05 22:46:57 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 217.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GBScleanR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:GBScleanR.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings GBScleanR_1.5.3.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/GBScleanR.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘GBScleanR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘GBScleanR’ version ‘1.5.3’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GBScleanR’ can be installed ... OK
* used C++ compiler: ‘g++ (GCC) 10.3.1’
* checking C++ specification ... NOTE
  Specified C++11: please drop specification unless essential
* checking installed package size ... NOTE
  installed size is  7.1Mb
  sub-directories of 1Mb or more:
    libs   6.0Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘BasicUsageOfGBScleanR.rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/GBScleanR.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GBScleanR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL GBScleanR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘GBScleanR’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (GCC) 10.3.1’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c gbsrCalcProb.cpp -o gbsrCalcProb.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c gbsrFB.cpp -o gbsrFB.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c gbsrIPO.cpp -o gbsrIPO.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c gbsrStats.cpp -o gbsrStats.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c gbsrViterbi.cpp -o gbsrViterbi.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c gbsrutil.cpp -o gbsrutil.o
g++ -std=gnu++11 -shared -L/home/biocbuild/R/R-4.3.0/lib -L/usr/local/lib -o GBScleanR.so RcppExports.o gbsrCalcProb.o gbsrFB.o gbsrIPO.o gbsrStats.o gbsrViterbi.o gbsrutil.o -L/home/biocbuild/R/R-4.3.0/lib -lR
installing to /home/biocbuild/R/R-4.3.0/site-library/00LOCK-GBScleanR/00new/GBScleanR/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GBScleanR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">GBScleanR.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(GBScleanR)
Loading required package: SeqArray
Loading required package: gdsfmt
&gt; 
&gt; test_check(&quot;GBScleanR&quot;)
Loading GDS file.
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb78663b6f.gds&#x27; (94.1K)
    # of fragments: 69
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb78663b6f.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb78663b6f.gds.tmp&#x27; (94.0K, reduced: 108B)
    # of fragments: 60
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb253e0b58.gds&#x27; (94.1K)
    # of fragments: 69
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb253e0b58.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb253e0b58.gds.tmp&#x27; (94.0K, reduced: 108B)
    # of fragments: 60
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb5189401c.gds&#x27; (94.1K)
    # of fragments: 69
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb5189401c.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb5189401c.gds.tmp&#x27; (94.0K, reduced: 108B)
    # of fragments: 60
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Mon Jun  5 22:46:09 2023
Variant Call Format (VCF) Import:
    file:
        sample.vcf (210.3K)
    file format: VCFv4.2
    genome reference: &lt;unknown&gt;
    # of sets of chromosomes (ploidy): 2
    # of samples: 102
    genotype field: GT
    genotype storage: bit2
    compression method: customized
    # of samples: 102
Output:
    /tmp/RtmpTiKrRa/sample35bbfb1860a83b.gds
Parsing &#x27;sample.vcf&#x27;:
+ genotype/data   { Bit2 2x102x242 ZIP_ra, 16B }
Digests:
    sample.id  [md5: 338086c89cac9760256e9d1ec0a77327]
    variant.id  [md5: 6f6b771cc6816e18766cd7b202765193]
    position  [md5: f3033fec247b8ec6980e81005e257bd8]
    chromosome  [md5: 891ee7d299e1dba9146b8ae33476741c]
    allele  [md5: a51167e967b708c016f5196aa910d180]
    genotype  [md5: b83af5eb9818d83c2ccaa40d494f15a8]
    phase  [md5: 9d686e01959b61df5fdc1a4684bd72b3]
    annotation/id  [md5: 2e4debeee6209485fcf7cb57d8e185e6]
    annotation/qual  [md5: 5a566f4332739a2b28d23b215163b70a]
    annotation/filter  [md5: cb74cdb22966d99a9290a2c804a10580]
    annotation/format/AD  [md5: 9c1d40375ae9a2d5fb0e3a5ef81f5c53]
Done.
Mon Jun  5 22:46:09 2023
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb1860a83b.gds&#x27; (52.6K)
    # of fragments: 108
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb1860a83b.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb1860a83b.gds.tmp&#x27; (51.9K, reduced: 648B)
    # of fragments: 54
Mon Jun  5 22:46:09 2023
Loading GDS file.
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb1860a83b.gds&#x27; (94.1K)
    # of fragments: 69
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb1860a83b.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb1860a83b.gds.tmp&#x27; (94.0K, reduced: 108B)
    # of fragments: 60
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb1c51ede8.gds&#x27; (94.1K)
    # of fragments: 69
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb1c51ede8.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb1c51ede8.gds.tmp&#x27; (94.0K, reduced: 108B)
    # of fragments: 60
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
The connection to the GDS file was closed.
Loading GDS file.
 &lt;None&gt; &lt;None&gt; &lt;None&gt; &lt;None&gt; &lt;None&gt; &lt;None&gt; &lt;None&gt; &lt;None&gt; &lt;None&gt; &lt;None&gt;The connection to the GDS file was closed.
Loading GDS file.
Reformatting FGT
The connection to the GDS file was closed.
Loading GDS file.
Reformatting FGT
The connection to the GDS file was closed.
Mon Jun  5 22:46:13 2023
Variant Call Format (VCF) Import:
    file:
        sample.vcf (210.3K)
    file format: VCFv4.2
    genome reference: &lt;unknown&gt;
    # of sets of chromosomes (ploidy): 2
    # of samples: 102
    genotype field: GT
    genotype storage: bit2
    compression method: customized
    # of samples: 102
Output:
    /tmp/RtmpTiKrRa/sample35bbfb34cba52.gds
Parsing &#x27;sample.vcf&#x27;:
+ genotype/data   { Bit2 2x102x242 ZIP_ra, 16B }
Digests:
    sample.id  [md5: 338086c89cac9760256e9d1ec0a77327]
    variant.id  [md5: 6f6b771cc6816e18766cd7b202765193]
    position  [md5: f3033fec247b8ec6980e81005e257bd8]
    chromosome  [md5: 891ee7d299e1dba9146b8ae33476741c]
    allele  [md5: a51167e967b708c016f5196aa910d180]
    genotype  [md5: b83af5eb9818d83c2ccaa40d494f15a8]
    phase  [md5: 9d686e01959b61df5fdc1a4684bd72b3]
    annotation/id  [md5: 2e4debeee6209485fcf7cb57d8e185e6]
    annotation/qual  [md5: 5a566f4332739a2b28d23b215163b70a]
    annotation/filter  [md5: cb74cdb22966d99a9290a2c804a10580]
    annotation/format/AD  [md5: 9c1d40375ae9a2d5fb0e3a5ef81f5c53]
Done.
Mon Jun  5 22:46:13 2023
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb34cba52.gds&#x27; (52.6K)
    # of fragments: 108
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb34cba52.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb34cba52.gds.tmp&#x27; (51.9K, reduced: 648B)
    # of fragments: 54
Mon Jun  5 22:46:13 2023
Loading GDS file.
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb34cba52.gds&#x27; (94.1K)
    # of fragments: 69
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb34cba52.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb34cba52.gds.tmp&#x27; (94.0K, reduced: 108B)
    # of fragments: 60
No parents info.
Mon Jun  5 22:46:13 2023
Variant Call Format (VCF) Import:
    file:
        out35bbfb3d014780.vcf (56.3K)
    file format: VCFv4.2
    genome reference: &lt;unknown&gt;
    # of sets of chromosomes (ploidy): 2
    # of samples: 51
    genotype field: GT
    genotype storage: bit2
    compression method: customized
    # of samples: 51
Output:
    /tmp/RtmpTiKrRa/newgds35bbfb60b71d78.gds
Parsing &#x27;out35bbfb3d014780.vcf&#x27;:
+ genotype/data   { Bit2 2x51x122 ZIP_ra, 16B }
Digests:
    sample.id  [md5: 4484844f13736eadbb3ddc714e1cf32a]
    variant.id  [md5: 012282239322483b7985e5e9f6da0686]
    position  [md5: 23839bd8366b957a39020507a515c47a]
    chromosome  [md5: 4457c426faf4c14ff1f038147e8a1394]
    allele  [md5: 24a0d22984288d9b0c5df216310185d9]
    genotype  [md5: 52594a7630cac724d6979f2445b85e77]
    phase  [md5: d57eb98f58a646ab75391b969884ee75]
    annotation/id  [md5: faaea825142a19bc382389720973bda7]
    annotation/qual  [md5: 1a83cd1cf317e4eddb6cf4d82085a846]
    annotation/filter  [md5: a3b02e52aea44205fec34a8e8bf47d7d]
    annotation/format/AD  [md5: ff2d47162f94676b604c9f9de692a555]
Done.
Mon Jun  5 22:46:13 2023
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/newgds35bbfb60b71d78.gds&#x27; (20.1K)
    # of fragments: 107
    save to &#x27;/tmp/RtmpTiKrRa/newgds35bbfb60b71d78.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/newgds35bbfb60b71d78.gds.tmp&#x27; (19.5K, reduced: 636B)
    # of fragments: 54
Mon Jun  5 22:46:13 2023
Loading GDS file.
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/newgds35bbfb60b71d78.gds&#x27; (31.0K)
    # of fragments: 69
    save to &#x27;/tmp/RtmpTiKrRa/newgds35bbfb60b71d78.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/newgds35bbfb60b71d78.gds.tmp&#x27; (30.9K, reduced: 108B)
    # of fragments: 60
No parents info.
Mon Jun  5 22:46:13 2023
Variant Call Format (VCF) Import:
    file:
        out35bbfb56c591ab.vcf (119.4K)
    file format: VCFv4.2
    genome reference: &lt;unknown&gt;
    # of sets of chromosomes (ploidy): 2
    # of samples: 51
    genotype field: GT
    genotype storage: bit2
    compression method: customized
    # of samples: 51
Output:
    /tmp/RtmpTiKrRa/newgds35bbfb4c252f8.gds
Parsing &#x27;out35bbfb56c591ab.vcf&#x27;:
+ genotype/data   { Bit2 2x51x122 ZIP_ra, 16B }
Digests:
    sample.id  [md5: 4484844f13736eadbb3ddc714e1cf32a]
    variant.id  [md5: 012282239322483b7985e5e9f6da0686]
    position  [md5: 23839bd8366b957a39020507a515c47a]
    chromosome  [md5: 4457c426faf4c14ff1f038147e8a1394]
    allele  [md5: 24a0d22984288d9b0c5df216310185d9]
    genotype  [md5: 52594a7630cac724d6979f2445b85e77]
    phase  [md5: d57eb98f58a646ab75391b969884ee75]
    annotation/id  [md5: faaea825142a19bc382389720973bda7]
    annotation/qual  [md5: 1a83cd1cf317e4eddb6cf4d82085a846]
    annotation/filter  [md5: a3b02e52aea44205fec34a8e8bf47d7d]
    annotation/format/AD  [md5: ff2d47162f94676b604c9f9de692a555]
    annotation/format/CFT  [md5: fec4208b14ccf4572a2eda8a929ea312]
    annotation/format/FAD  [md5: 43f0dc39fa2292c4bda8101961355d2f]
    annotation/format/FGT  [md5: 2fb19a3ea5824e116fcfba82896333ab]
Done.
Mon Jun  5 22:46:13 2023
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/newgds35bbfb4c252f8.gds&#x27; (30.8K)
    # of fragments: 140
    save to &#x27;/tmp/RtmpTiKrRa/newgds35bbfb4c252f8.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/newgds35bbfb4c252f8.gds.tmp&#x27; (29.9K, reduced: 852B)
    # of fragments: 69
Mon Jun  5 22:46:13 2023
Loading GDS file.
Reformatting FGT
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Working on &#x27;annotation/format/FAD&#x27; ...
Working on &#x27;annotation/format/FGT&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/newgds35bbfb4c252f8.gds&#x27; (51.7K)
    # of fragments: 91
    save to &#x27;/tmp/RtmpTiKrRa/newgds35bbfb4c252f8.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/newgds35bbfb4c252f8.gds.tmp&#x27; (50.3K, reduced: 1.4K)
    # of fragments: 79
The connection to the GDS file was closed.
The connection to the GDS file was closed.
Loading GDS file.
Working on &#x27;genotype&#x27; ...
Working on &#x27;phase&#x27; ...
Working on &#x27;annotation/format/AD&#x27; ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpTiKrRa/sample35bbfb269c361.gds&#x27; (94.1K)
    # of fragments: 69
    save to &#x27;/tmp/RtmpTiKrRa/sample35bbfb269c361.gds.tmp&#x27;
    rename &#x27;/tmp/RtmpTiKrRa/sample35bbfb269c361.gds.tmp&#x27; (94.0K, reduced: 108B)
    # of fragments: 60
As `mating` was not specified, set the following mating design.
     [,1]
[1,]    3
[2,]    3
Member IDs were not assigned to samples.
Assign 4 to all samples as member ID.
Set the number of threads: 1
Start cleaning...

Now cleaning chr 1...
Cycle 1: 
Forward round of genotype estimation ...
Founder genotype probability calculation ...                                                                      Founder genotype probability calculation at marker#: 10                                                                      Founder genotype probability calculation at marker#: 20                                                                      Founder genotype probability calculation at marker#: 30                                                                      Founder genotype probability calculation at marker#: 40                                                                      Founder genotype probability calculation at marker#: 50                                                                      Founder genotype probability calculation at marker#: 60                                                                      Founder genotype probability calculation at marker#: 70                                                                      Founder genotype probability calculation at marker#: 80                                                                      Founder genotype probability calculation at marker#: 90                                                                      Founder genotype probability calculation at marker#: 100                                                                      Founder genotype probability calculation at marker#: 110                                                                      Founder genotype probability calculation at marker#: 120                                                                      Founder genotype probability calculation at marker#: 130                                                                      Founder genotype probability calculation at marker#: 140                                                                      Founder genotype probability calculation at marker#: 150                                                                      Founder genotype probability calculation at marker#: 160                                                                      Founder genotype probability calculation at marker#: 170                                                                      Founder genotype probability calculation at marker#: 180                                                                      Founder genotype probability calculation at marker#: 190                                                                      Founder genotype probability calculation at marker#: 200                                                                      Founder genotype probability calculation at marker#: 210                                                                      Founder genotype probability calculation at marker#: 220                                                                      Founder genotype probability calculation at marker#: 230                                                                      Founder genotype probability calculation at marker#: 240                                                                      Backtracking best genotype sequences at marker#: 240                                                                      Backtracking best genotype sequences at marker#: 230                                                                      Backtracking best genotype sequences at marker#: 220                                                                      Backtracking best genotype sequences at marker#: 210                                                                      Backtracking best genotype sequences at marker#: 200                                                                      Backtracking best genotype sequences at marker#: 190                                                                      Backtracking best genotype sequences at marker#: 180                                                                      Backtracking best genotype sequences at marker#: 170                                                                      Backtracking best genotype sequences at marker#: 160                                                                      Backtracking best genotype sequences at marker#: 150                                                                      Backtracking best genotype sequences at marker#: 140                                                                      Backtracking best genotype sequences at marker#: 130                                                                      Backtracking best genotype sequences at marker#: 120                                                                      Backtracking best genotype sequences at marker#: 110                                                                      Backtracking best genotype sequences at marker#: 100                                                                      Backtracking best genotype sequences at marker#: 90                                                                      Backtracking best genotype sequences at marker#: 80                                                                      Backtracking best genotype sequences at marker#: 70                                                                      Backtracking best genotype sequences at marker#: 60                                                                      Backtracking best genotype sequences at marker#: 50                                                                      Backtracking best genotype sequences at marker#: 40                                                                      Backtracking best genotype sequences at marker#: 30                                                                      Backtracking best genotype sequences at marker#: 20                                                                      Backtracking best genotype sequences at marker#: 10                                                                      Backtracking best genotype sequences: Done!                                                                      Offspring genotype probability calculation ...                                                                                                                                            Backward round of genotype estimation  ...
Founder genotype probability calculation ...                                                                      Founder genotype probability calculation at marker#: 10                                                                      Founder genotype probability calculation at marker#: 20                                                                      Founder genotype probability calculation at marker#: 30                                                                      Founder genotype probability calculation at marker#: 40                                                                      Founder genotype probability calculation at marker#: 50                                                                      Founder genotype probability calculation at marker#: 60                                                                      Founder genotype probability calculation at marker#: 70                                                                      Founder genotype probability calculation at marker#: 80                                                                      Founder genotype probability calculation at marker#: 90                                                                      Founder genotype probability calculation at marker#: 100                                                                      Founder genotype probability calculation at marker#: 110                                                                      Founder genotype probability calculation at marker#: 120                                                                      Founder genotype probability calculation at marker#: 130                                                                      Founder genotype probability calculation at marker#: 140                                                                      Founder genotype probability calculation at marker#: 150                                                                      Founder genotype probability calculation at marker#: 160                                                                      Founder genotype probability calculation at marker#: 170                                                                      Founder genotype probability calculation at marker#: 180                                                                      Founder genotype probability calculation at marker#: 190                                                                      Founder genotype probability calculation at marker#: 200                                                                      Founder genotype probability calculation at marker#: 210                                                                      Founder genotype probability calculation at marker#: 220                                                                      Founder genotype probability calculation at marker#: 230                                                                      Founder genotype probability calculation at marker#: 240                                                                      Backtracking best genotype sequences at marker#: 240                                                                      Backtracking best genotype sequences at marker#: 230                                                                      Backtracking best genotype sequences at marker#: 220                                                                      Backtracking best genotype sequences at marker#: 210                                                                      Backtracking best genotype sequences at marker#: 200                                                                      Backtracking best genotype sequences at marker#: 190                                                                      Backtracking best genotype sequences at marker#: 180                                                                      Backtracking best genotype sequences at marker#: 170                                                                      Backtracking best genotype sequences at marker#: 160                                                                      Backtracking best genotype sequences at marker#: 150                                                                      Backtracking best genotype sequences at marker#: 140                                                                      Backtracking best genotype sequences at marker#: 130                                                                      Backtracking best genotype sequences at marker#: 120                                                                      Backtracking best genotype sequences at marker#: 110                                                                      Backtracking best genotype sequences at marker#: 100                                                                      Backtracking best genotype sequences at marker#: 90                                                                      Backtracking best genotype sequences at marker#: 80                                                                      Backtracking best genotype sequences at marker#: 70                                                                      Backtracking best genotype sequences at marker#: 60                                                                      Backtracking best genotype sequences at marker#: 50                                                                      Backtracking best genotype sequences at marker#: 40                                                                      Backtracking best genotype sequences at marker#: 30                                                                      Backtracking best genotype sequences at marker#: 20                                                                      Backtracking best genotype sequences at marker#: 10                                                                      Backtracking best genotype sequences: Done!                                                                      Offspring genotype probability calculation ...                                                                                                                                            Paramter optimization ...
Cycle 2: 
Forward round of genotype estimation ...
Founder genotype probability calculation ...                                                                      Founder genotype probability calculation at marker#: 10                                                                      Founder genotype probability calculation at marker#: 20                                                                      Founder genotype probability calculation at marker#: 30                                                                      Founder genotype probability calculation at marker#: 40                                                                      Founder genotype probability calculation at marker#: 50                                                                      Founder genotype probability calculation at marker#: 60                                                                      Founder genotype probability calculation at marker#: 70                                                                      Founder genotype probability calculation at marker#: 80                                                                      Founder genotype probability calculation at marker#: 90                                                                      Founder genotype probability calculation at marker#: 100                                                                      Founder genotype probability calculation at marker#: 110                                                                      Founder genotype probability calculation at marker#: 120                                                                      Founder genotype probability calculation at marker#: 130                                                                      Founder genotype probability calculation at marker#: 140                                                                      Founder genotype probability calculation at marker#: 150                                                                      Founder genotype probability calculation at marker#: 160                                                                      Founder genotype probability calculation at marker#: 170                                                                      Founder genotype probability calculation at marker#: 180                                                                      Founder genotype probability calculation at marker#: 190                                                                      Founder genotype probability calculation at marker#: 200                                                                      Founder genotype probability calculation at marker#: 210                                                                      Founder genotype probability calculation at marker#: 220                                                                      Founder genotype probability calculation at marker#: 230                                                                      Founder genotype probability calculation at marker#: 240                                                                      Backtracking best genotype sequences at marker#: 240                                                                      Backtracking best genotype sequences at marker#: 230                                                                      Backtracking best genotype sequences at marker#: 220                                                                      Backtracking best genotype sequences at marker#: 210                                                                      Backtracking best genotype sequences at marker#: 200                                                                      Backtracking best genotype sequences at marker#: 190                                                                      Backtracking best genotype sequences at marker#: 180                                                                      Backtracking best genotype sequences at marker#: 170                                                                      Backtracking best genotype sequences at marker#: 160                                                                      Backtracking best genotype sequences at marker#: 150                                                                      Backtracking best genotype sequences at marker#: 140                                                                      Backtracking best genotype sequences at marker#: 130                                                                      Backtracking best genotype sequences at marker#: 120                                                                      Backtracking best genotype sequences at marker#: 110                                                                      Backtracking best genotype sequences at marker#: 100                                                                      Backtracking best genotype sequences at marker#: 90                                                                      Backtracking best genotype sequences at marker#: 80                                                                      Backtracking best genotype sequences at marker#: 70                                                                      Backtracking best genotype sequences at marker#: 60                                                                      Backtracking best genotype sequences at marker#: 50                                                                      Backtracking best genotype sequences at marker#: 40                                                                      Backtracking best genotype sequences at marker#: 30                                                                      Backtracking best genotype sequences at marker#: 20                                                                      Backtracking best genotype sequences at marker#: 10                                                                      Backtracking best genotype sequences: Done!                                                                      Offspring genotype probability calculation ...                                                                                                                                            Backward round of genotype estimation  ...
Founder genotype probability calculation ...                                                                      Founder genotype probability calculation at marker#: 10                                                                      Founder genotype probability calculation at marker#: 20                                                                      Founder genotype probability calculation at marker#: 30                                                                      Founder genotype probability calculation at marker#: 40                                                                      Founder genotype probability calculation at marker#: 50                                                                      Founder genotype probability calculation at marker#: 60                                                                      Founder genotype probability calculation at marker#: 70                                                                      Founder genotype probability calculation at marker#: 80                                                                      Founder genotype probability calculation at marker#: 90                                                                      Founder genotype probability calculation at marker#: 100                                                                      Founder genotype probability calculation at marker#: 110                                                                      Founder genotype probability calculation at marker#: 120                                                                      Founder genotype probability calculation at marker#: 130                                                                      Founder genotype probability calculation at marker#: 140                                                                      Founder genotype probability calculation at marker#: 150                                                                      Founder genotype probability calculation at marker#: 160                                                                      Founder genotype probability calculation at marker#: 170                                                                      Founder genotype probability calculation at marker#: 180                                                                      Founder genotype probability calculation at marker#: 190                                                                      Founder genotype probability calculation at marker#: 200                                                                      Founder genotype probability calculation at marker#: 210                                                                      Founder genotype probability calculation at marker#: 220                                                                      Founder genotype probability calculation at marker#: 230                                                                      Founder genotype probability calculation at marker#: 240                                                                      Backtracking best genotype sequences at marker#: 240                                                                      Backtracking best genotype sequences at marker#: 230                                                                      Backtracking best genotype sequences at marker#: 220                                                                      Backtracking best genotype sequences at marker#: 210                                                                      Backtracking best genotype sequences at marker#: 200                                                                      Backtracking best genotype sequences at marker#: 190                                                                      Backtracking best genotype sequences at marker#: 180                                                                      Backtracking best genotype sequences at marker#: 170                                                                      Backtracking best genotype sequences at marker#: 160                                                                      Backtracking best genotype sequences at marker#: 150                                                                      Backtracking best genotype sequences at marker#: 140                                                                      Backtracking best genotype sequences at marker#: 130                                                                      Backtracking best genotype sequences at marker#: 120                                                                      Backtracking best genotype sequences at marker#: 110                                                                      Backtracking best genotype sequences at marker#: 100                                                                      Backtracking best genotype sequences at marker#: 90                                                                      Backtracking best genotype sequences at marker#: 80                                                                      Backtracking best genotype sequences at marker#: 70                                                                      Backtracking best genotype sequences at marker#: 60                                                                      Backtracking best genotype sequences at marker#: 50                                                                      Backtracking best genotype sequences at marker#: 40                                                                      Backtracking best genotype sequences at marker#: 30                                                                      Backtracking best genotype sequences at marker#: 20                                                                      Backtracking best genotype sequences at marker#: 10                                                                      Backtracking best genotype sequences: Done!                                                                      Offspring genotype probability calculation ...                                                                                                                                            Summarizing output ...
Done!
The connection to the GDS file was closed.
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 271 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 15.204   2.213  17.466 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GBScleanR.Rcheck/GBScleanR-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>GbsrGenotypeData-class</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GbsrScheme-class</TD><TD style="text-align: right;">0.07</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.07</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addScheme</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assignScheme</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>boxplotGBSR</TD><TD style="text-align: right;">0.608</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.612</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>closeGDS</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countGenotype</TD><TD style="text-align: right;">0.362</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.399</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countRead</TD><TD style="text-align: right;">0.406</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.431</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>estGeno</TD><TD style="text-align: right;">1.873</TD><TD style="text-align: right;">0.532</TD><TD style="text-align: right;">2.432</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gbsrGDS2CSV</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gbsrGDS2VCF</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gbsrVCF2GDS</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.103</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getAllele</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getChromosome</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountAlleleAlt</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountAlleleMissing</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountAlleleRef</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountGenoAlt</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountGenoHet</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountGenoMissing</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountGenoRef</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountRead</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountReadAlt</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCountReadRef</TD><TD style="text-align: right;">0.03</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.03</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getGenotype</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getHaplotype</TD><TD style="text-align: right;">1.262</TD><TD style="text-align: right;">0.540</TD><TD style="text-align: right;">1.828</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getInfo</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMAC</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMAF</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMarID</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMeanReadAlt</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMeanReadRef</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMedianReadAlt</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMedianReadRef</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getParents</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPosition</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getRead</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSDReadAlt</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSDReadRef</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSamID</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>histGBSR</TD><TD style="text-align: right;">0.349</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.350</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>initScheme</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isOpenGDS</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadGDS</TD><TD style="text-align: right;">0.086</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.089</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nmar</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nsam</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pairsGBSR</TD><TD style="text-align: right;">0.210</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.217</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotDosage</TD><TD style="text-align: right;">0.260</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.288</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotGBSR</TD><TD style="text-align: right;">0.293</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.297</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotReadRatio</TD><TD style="text-align: right;">0.233</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.242</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reopenGDS</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>resetCallFilter</TD><TD style="text-align: right;">0.343</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">0.415</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>resetFilter</TD><TD style="text-align: right;">0.390</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">0.483</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>resetMarFilter</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>resetSamFilter</TD><TD style="text-align: right;">0.109</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.118</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setCallFilter</TD><TD style="text-align: right;">0.655</TD><TD style="text-align: right;">0.147</TD><TD style="text-align: right;">0.803</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setInfoFilter</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setMarFilter</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setParents</TD><TD style="text-align: right;">1.777</TD><TD style="text-align: right;">0.574</TD><TD style="text-align: right;">2.381</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setReplicates</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setSamFilter</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>showScheme</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>thinMarker</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>validMar</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>validSam</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
