<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for Biostrings on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><B>[<A href="..#B">B</A>]</B></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-25 05:45:51 -0000 (Thu, 25 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4223</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for Biostrings on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the Biostrings package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/Biostrings.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>212</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">Biostrings</A>&nbsp;2.69.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/Biostrings">landing page</A>)</SPAN><BR>Hervé Pagès<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-24&nbsp;15:28:50&nbsp;-0000&nbsp;(Wed,&nbsp;24&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/Biostrings</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">03aa8df</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-05-16&nbsp;05:13:39&nbsp;-0000&nbsp;(Tue,&nbsp;16&nbsp;May&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: Biostrings</TD></TR>
<TR><TD><B>Version</B>: 2.69.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:Biostrings.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings Biostrings_2.69.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-25 03:52:51 -0000 (Thu, 25 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-25 04:06:11 -0000 (Thu, 25 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 800.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: Biostrings.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 2</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:Biostrings.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings Biostrings_2.69.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/Biostrings.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘Biostrings/DESCRIPTION’ ... OK
* this is package ‘Biostrings’ version ‘2.69.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘Rmpi’
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘Biostrings’ can be installed ... OK
* used C compiler: ‘gcc (GCC) 10.3.1’
* checking installed package size ... NOTE
  installed size is 14.8Mb
  sub-directories of 1Mb or more:
    R         1.7Mb
    extdata  11.1Mb
    libs      1.1Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘methods’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) AAString-class.Rd:74-78: \item in \describe must have non-empty label
checkRd: (5) AAString-class.Rd:86-92: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:87-90: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:91-96: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:97-99: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:100-103: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:104-107: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:108-111: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:112-116: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:117-121: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:122-125: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:126-129: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:130-133: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:134-137: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:138-141: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:147-151: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:152-156: \item in \describe must have non-empty label
checkRd: (5) DNAString-class.Rd:57-61: \item in \describe must have non-empty label
checkRd: (5) DNAString-class.Rd:69-75: \item in \describe must have non-empty label
checkRd: (5) InDel-class.Rd:32-35: \item in \describe must have non-empty label
checkRd: (5) InDel-class.Rd:36-39: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:49-52: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:53-56: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:57-61: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:62-66: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:67-71: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:79-83: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:91-99: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:110-115: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:116-119: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:113-116: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:117-122: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:123-140: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:141-145: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:146-152: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:160-164: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:165-168: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:169-172: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:180-185: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:193-197: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:198-203: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:157-161: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:162-167: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:168-178: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:179-189: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:190-200: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:201-216: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:217-220: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:221-224: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:225-228: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:229-232: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:233-236: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:237-240: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:241-244: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:245-249: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:250-253: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:261-267: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:268-274: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:275-282: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:290-298: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:299-304: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:305-312: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:313-320: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:321-337: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:241-244: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:245-249: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:250-253: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:254-257: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:258-261: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:262-269: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:270-273: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:282-285: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:294-297: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:298-301: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:154-160: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:161-165: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:166-169: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:177-180: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:181-187: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:188-193: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:202-210: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:211-214: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:215-218: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:219-222: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:230-234: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:235-238: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:239-243: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:244-250: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:258-262: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:263-268: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:269-274: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:275-279: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:280-283: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:284-289: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:295-299: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:300-304: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:111-114: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:115-119: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:120-123: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:124-129: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:130-133: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:143-146: \item in \describe must have non-empty label
checkRd: (5) RNAString-class.Rd:55-59: \item in \describe must have non-empty label
checkRd: (5) RNAString-class.Rd:67-73: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:94-98: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:106-111: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:112-115: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:123-126: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:127-130: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:138-148: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:156-169: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:170-173: \item in \describe must have non-empty label
checkRd: (5) XStringPartialMatches-class.Rd:32-35: \item in \describe must have non-empty label
checkRd: (5) XStringPartialMatches-class.Rd:36-39: \item in \describe must have non-empty label
checkRd: (5) XStringPartialMatches-class.Rd:49-54: \item in \describe must have non-empty label
checkRd: (-1) XStringQuality-class.Rd:86: Escaped LaTeX specials: \#
checkRd: (5) XStringQuality-class.Rd:106-109: \item in \describe must have non-empty label
checkRd: (5) XStringQuality-class.Rd:111-115: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:116-118: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:119-121: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:122-124: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:135-138: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:139-145: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:146-155: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:156-162: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:163-166: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:176-193: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:194-204: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:205-215: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:216-221: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:231-234: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:235-238: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:239-242: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:251-254: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:255-258: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:259-262: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:263-266: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:275-280: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:281-286: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:287-290: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:291-298: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:299-302: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:303-311: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:55-65: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:66-72: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:80-85: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:86-90: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:91-94: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:95-98: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:106-109: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:110-113: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:122-126: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:127-131: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:141-143: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:144-146: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:73-77: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:87-93: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:103-117: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:118-121: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:122-131: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:136-143: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:144-147: \item in \describe must have non-empty label
checkRd: (5) injectHardMask.Rd:58-61: \item in \describe must have non-empty label
checkRd: (5) injectHardMask.Rd:62-70: \item in \describe must have non-empty label
checkRd: (5) injectHardMask.Rd:71-74: \item in \describe must have non-empty label
checkRd: (-1) matchLRPatterns.Rd:65: Escaped LaTeX specials: \_ \_
checkRd: (-1) matchProbePair.Rd:53: Escaped LaTeX specials: \_ \_
checkRd: (-1) matchProbePair.Rd:54: Escaped LaTeX specials: \_ \_
checkRd: (-1) matchProbePair.Rd:55: Escaped LaTeX specials: \_ \_
checkRd: (-1) matchProbePair.Rd:56: Escaped LaTeX specials: \_ \_
checkRd: (-1) phiX174Phage.Rd:18: Escaped LaTeX specials: \_
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘strsplit’ ‘twoWayAlphabetFrequency’
Undocumented S4 methods:
  generic &#x27;match&#x27; and siglist &#x27;Vector,XStringSet&#x27;
  generic &#x27;match&#x27; and siglist &#x27;XStringSet,Vector&#x27;
  generic &#x27;match&#x27; and siglist &#x27;XStringSet,vector&#x27;
  generic &#x27;match&#x27; and siglist &#x27;vector,XStringSet&#x27;
  generic &#x27;parallel_slot_names&#x27; and siglist &#x27;ByPos_MIndex&#x27;
  generic &#x27;parallel_slot_names&#x27; and siglist &#x27;MIndex&#x27;
  generic &#x27;pcompare&#x27; and siglist &#x27;Vector,XStringSet&#x27;
  generic &#x27;pcompare&#x27; and siglist &#x27;XStringSet,Vector&#x27;
  generic &#x27;pcompare&#x27; and siglist &#x27;XStringSet,vector&#x27;
  generic &#x27;pcompare&#x27; and siglist &#x27;vector,XStringSet&#x27;
  generic &#x27;relistToClass&#x27; and siglist &#x27;XString&#x27;
  generic &#x27;strsplit&#x27; and siglist &#x27;XStringSet&#x27;
  generic &#x27;twoWayAlphabetFrequency&#x27; and siglist &#x27;XString,XString&#x27;
  generic &#x27;twoWayAlphabetFrequency&#x27; and siglist &#x27;XString,XStringSet&#x27;
  generic &#x27;twoWayAlphabetFrequency&#x27; and siglist &#x27;XStringSet,XString&#x27;
  generic &#x27;twoWayAlphabetFrequency&#x27; and siglist &#x27;XStringSet,XStringSet&#x27;
  generic &#x27;unstrsplit&#x27; and siglist &#x27;XStringSet&#x27;
  generic &#x27;unstrsplit&#x27; and siglist &#x27;XStringSetList&#x27;
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                      user system elapsed
matchPDict-exact   363.060  1.130 364.819
matchPDict-inexact  46.080  0.284  46.431
findPalindromes     38.540  0.055  38.646
XStringSet-class    14.315  0.515  14.871
XStringSet-io        7.501  0.248   7.764
matchPattern         5.861  0.116   5.985
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘run_unitTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘matchprobes.Rmd’ using ‘UTF-8’... OK
  ‘Biostrings2Classes.Rnw’ using ‘UTF-8’... OK
  ‘BiostringsQuickOverview.Rnw’ using ‘UTF-8’... OK
  ‘MultipleAlignments.Rnw’ using ‘UTF-8’... OK
  ‘PairwiseAlignments.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 WARNINGs, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/Biostrings.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">Biostrings.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL Biostrings
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘Biostrings’ ...
** using staged installation
** libs
using C compiler: ‘gcc (GCC) 10.3.1’
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c BAB_class.c -o BAB_class.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c BitMatrix.c -o BitMatrix.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c IRanges_stubs.c -o IRanges_stubs.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c MIndex_class.c -o MIndex_class.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c PreprocessedTB_class.c -o PreprocessedTB_class.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c R_init_Biostrings.c -o R_init_Biostrings.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c RoSeqs_utils.c -o RoSeqs_utils.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c S4Vectors_stubs.c -o S4Vectors_stubs.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c SparseList_utils.c -o SparseList_utils.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c XStringSetList_class.c -o XStringSetList_class.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c XStringSet_class.c -o XStringSet_class.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c XString_class.c -o XString_class.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c XVector_stubs.c -o XVector_stubs.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c align_needwunsQS.c -o align_needwunsQS.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c align_pairwiseAlignment.c -o align_pairwiseAlignment.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c align_utils.c -o align_utils.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c find_palindromes.c -o find_palindromes.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c gtestsim.c -o gtestsim.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c inject_code.c -o inject_code.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c letter_frequency.c -o letter_frequency.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c lowlevel_matching.c -o lowlevel_matching.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_PWM.c -o match_PWM.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_pattern.c -o match_pattern.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_pattern_boyermoore.c -o match_pattern_boyermoore.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_pattern_indels.c -o match_pattern_indels.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_pattern_shiftor.c -o match_pattern_shiftor.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_pdict.c -o match_pdict.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_pdict_ACtree2.c -o match_pdict_ACtree2.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_pdict_Twobit.c -o match_pdict_Twobit.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_pdict_utils.c -o match_pdict_utils.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c match_reporting.c -o match_reporting.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c matchprobes.c -o matchprobes.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c pmatchPattern.c -o pmatchPattern.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c read_fasta_files.c -o read_fasta_files.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c read_fastq_files.c -o read_fastq_files.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c replaceAt.c -o replaceAt.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c replace_letter_at.c -o replace_letter_at.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c strutils.c -o strutils.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c translate.c -o translate.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c unstrsplit_methods.c -o unstrsplit_methods.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c utils.c -o utils.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/IRanges/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c xscat.c -o xscat.o
gcc -shared -L/home/biocbuild/R/R-4.3.0/lib -L/usr/local/lib -o Biostrings.so BAB_class.o BitMatrix.o IRanges_stubs.o MIndex_class.o PreprocessedTB_class.o R_init_Biostrings.o RoSeqs_utils.o S4Vectors_stubs.o SparseList_utils.o XStringSetList_class.o XStringSet_class.o XString_class.o XVector_stubs.o align_needwunsQS.o align_pairwiseAlignment.o align_utils.o find_palindromes.o gtestsim.o inject_code.o letter_frequency.o lowlevel_matching.o match_PWM.o match_pattern.o match_pattern_boyermoore.o match_pattern_indels.o match_pattern_shiftor.o match_pdict.o match_pdict_ACtree2.o match_pdict_Twobit.o match_pdict_utils.o match_reporting.o matchprobes.o pmatchPattern.o read_fasta_files.o read_fastq_files.o replaceAt.o replace_letter_at.o strutils.o translate.o unstrsplit_methods.o utils.o xscat.o -L/home/biocbuild/R/R-4.3.0/lib -lR
installing to /home/biocbuild/R/R-4.3.0/site-library/00LOCK-Biostrings/00new/Biostrings/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
in method for ‘twoWayAlphabetFrequencyByQuality’ with signature ‘&quot;QualityScaledXStringSet&quot;’: no definition for class “QualityScaledXStringSet”
Creating a new generic function for ‘strsplit’ in package ‘Biostrings’
Creating a generic function for ‘ls’ from package ‘base’ in package ‘Biostrings’
Creating a new generic function for ‘pattern’ in package ‘Biostrings’
Creating a new generic function for ‘offset’ in package ‘Biostrings’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (Biostrings)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">Biostrings.Rcheck/tests/run_unitTests.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;Biostrings&quot;) || stop(&quot;unable to load Biostrings package&quot;)
Loading required package: Biostrings
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: XVector
Loading required package: GenomeInfoDb

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

[1] TRUE
&gt; Biostrings:::.test()


RUNIT TEST PROTOCOL -- Thu May 25 04:03:46 2023 
*********************************************** 
Number of test functions: 41 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
Biostrings RUnit Tests - 41 test functions, 0 errors, 0 failures
Number of test functions: 41 
Number of errors: 0 
Number of failures: 0 
Warning messages:
1: In XStringSet(&quot;DNA&quot;, x, start = start, end = end, width = width,  :
  metadata columns on input DNAStringSet object were dropped
2: In XStringSet(&quot;DNA&quot;, x, start = start, end = end, width = width,  :
  metadata columns on input DNAStringSet object were dropped
3: In XStringSet(&quot;DNA&quot;, x, start = start, end = end, width = width,  :
  metadata columns on input DNAStringSet object were dropped
&gt; 
&gt; proc.time()
   user  system elapsed 
 12.465   0.367  12.857 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">Biostrings.Rcheck/Biostrings-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>AAString-class</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AMINO_ACID_CODE</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AlignedXStringSet-class</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DNAString-class</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GENETIC_CODE</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HNF4alpha</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IUPAC_CODE_MAP</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.185</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MIndex-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MaskedXString-class</TD><TD style="text-align: right;">0.239</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.275</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MultipleAlignment-class</TD><TD style="text-align: right;">1.591</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">1.603</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PDict-class</TD><TD style="text-align: right;">4.636</TD><TD style="text-align: right;">0.119</TD><TD style="text-align: right;">4.769</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PairwiseAlignments-class</TD><TD style="text-align: right;">0.772</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.790</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PairwiseAlignments-io</TD><TD style="text-align: right;">2.959</TD><TD style="text-align: right;">0.127</TD><TD style="text-align: right;">3.091</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QualityScaledXStringSet-class</TD><TD style="text-align: right;">0.181</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.187</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RNAString-class</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XString-class</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringQuality-class</TD><TD style="text-align: right;">0.19</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.19</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringSet-class</TD><TD style="text-align: right;">14.315</TD><TD style="text-align: right;"> 0.515</TD><TD style="text-align: right;">14.871</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringSet-comparison</TD><TD style="text-align: right;">3.378</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">3.426</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringSet-io</TD><TD style="text-align: right;">7.501</TD><TD style="text-align: right;">0.248</TD><TD style="text-align: right;">7.764</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringSetList-class</TD><TD style="text-align: right;">0.315</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.319</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringViews-class</TD><TD style="text-align: right;">0.147</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.150</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>align-utils</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>chartr</TD><TD style="text-align: right;">0.722</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.743</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>detail</TD><TD style="text-align: right;">0.368</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.377</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dinucleotideFrequencyTest</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findPalindromes</TD><TD style="text-align: right;">38.540</TD><TD style="text-align: right;"> 0.055</TD><TD style="text-align: right;">38.646</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSeq</TD><TD style="text-align: right;">0.063</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gregexpr2</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>injectHardMask</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>letter</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>letterFrequency</TD><TD style="text-align: right;">0.998</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">1.068</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>longestConsecutive</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lowlevel-matching</TD><TD style="text-align: right;">0.638</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.650</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>maskMotif</TD><TD style="text-align: right;">1.512</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.537</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>match-utils</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>matchLRPatterns</TD><TD style="text-align: right;">0.651</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.677</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>matchPDict-exact</TD><TD style="text-align: right;">363.060</TD><TD style="text-align: right;">  1.130</TD><TD style="text-align: right;">364.819</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>matchPDict-inexact</TD><TD style="text-align: right;">46.080</TD><TD style="text-align: right;"> 0.284</TD><TD style="text-align: right;">46.431</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>matchPWM</TD><TD style="text-align: right;">2.545</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.565</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>matchPattern</TD><TD style="text-align: right;">5.861</TD><TD style="text-align: right;">0.116</TD><TD style="text-align: right;">5.985</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>matchProbePair</TD><TD style="text-align: right;">1.632</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.643</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>matchprobes</TD><TD style="text-align: right;">0.267</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.280</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>misc</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>needwunsQS</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nucleotideFrequency</TD><TD style="text-align: right;">0.774</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.787</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>padAndClip</TD><TD style="text-align: right;">0.625</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.645</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pairwiseAlignment</TD><TD style="text-align: right;">0.855</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.872</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phiX174Phage</TD><TD style="text-align: right;">0.534</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.539</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pid</TD><TD style="text-align: right;">0.500</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.501</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>replaceAt</TD><TD style="text-align: right;">2.807</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">2.811</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>replaceLetterAt</TD><TD style="text-align: right;">0.906</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.919</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reverseComplement</TD><TD style="text-align: right;">1.385</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">1.403</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqinfo-methods</TD><TD style="text-align: right;">0.743</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.748</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>stringDist</TD><TD style="text-align: right;">3.961</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">3.974</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>substitution_matrices</TD><TD style="text-align: right;">1.161</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.175</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>toComplex</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>translate</TD><TD style="text-align: right;">1.576</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.587</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>trimLRPatterns</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>xscat</TD><TD style="text-align: right;">1.416</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.425</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>yeastSEQCHR1</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
