<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - BUILD results for qvalue on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><B>[<A href="..#Q">Q</A>]</B></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-24 01:46:19 -0000 (Wed, 24 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4131</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">BUILD results for qvalue on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the qvalue package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/qvalue.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1587</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc selected">BUILD</TD><TD class="STAGE checksrc">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">qvalue</A>&nbsp;2.33.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/qvalue">landing page</A>)</SPAN><BR>John D. Storey <jstorey@princeton.edu>, Andrew J. Bass<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-22&nbsp;17:00:18&nbsp;-0000&nbsp;(Mon,&nbsp;22&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/qvalue</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">5430b4a</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;13:39:50&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc selected"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc"><SPAN class="glyph skipped">skipped</SPAN></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: qvalue</TD></TR>
<TR><TD><B>Version</B>: 2.33.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data qvalue</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-23 12:35:58 -0000 (Tue, 23 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-23 12:36:09 -0000 (Tue, 23 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 11.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>PackageFile</B>: None</TD></TR>
<TR><TD><B>PackageFileSize</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data qvalue
###
##############################################################################
##############################################################################


* checking for file ‘qvalue/DESCRIPTION’ ... OK
* preparing ‘qvalue’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘qvalue.Rnw’ using knitr

		Information on package &#x27;qvalue&#x27;

Description:

Package:             qvalue
Type:                Package
Title:               Q-value estimation
                     for false discovery
                     rate control
Version:             2.33.0
Date:                2015-03-24
Authors@R:           as.person(c( &quot;John
                     D. Storey
                     &lt;jdstorey@princeton.edu&gt;
                     [aut, cre]&quot;, &quot;Andrew
                     J. Bass
                     &lt;ajbass@emory.edu&gt;
                     [aut]&quot;, &quot;Alan Dabney
                     [aut]&quot;, &quot;David
                     Robinson [aut]&quot;,
                     &quot;Gregory Warnes
                     [ctb]&quot; ))
Maintainer:          John D. Storey
                     &lt;jstorey@princeton.edu&gt;,
                     Andrew J. Bass
                     &lt;ajbass@emory.edu&gt;
biocViews:           MultipleComparisons
Description:         This package takes a
                     list of p-values
                     resulting from the
                     simultaneous testing
                     of many hypotheses
                     and estimates their
                     q-values and local
                     FDR values. The
                     q-value of a test
                     measures the
                     proportion of false
                     positives incurred
                     (called the false
                     discovery rate) when
                     that particular test
                     is called
                     significant.  The
                     local FDR measures
                     the posterior
                     probability the null
                     hypothesis is true
                     given the test&#x27;s
                     p-value. Various
                     plots are
                     automatically
                     generated, allowing
                     one to make sensible
                     significance
                     cut-offs. Several
                     mathematical results
                     have recently been
                     shown on the
                     conservative
                     accuracy of the
                     estimated q-values
                     from this software.
                     The software can be
                     applied to problems
                     in genomics, brain
                     imaging,
                     astrophysics, and
                     data mining.
VignetteBuilder:     knitr
Imports:             splines, ggplot2,
                     grid, reshape2
Suggests:            knitr
Depends:             R(&gt;= 2.10)
URL:                 http://github.com/jdstorey/qvalue
License:             LGPL
RoxygenNote:         5.0.1
git_url:             https://git.bioconductor.org/packages/qvalue
git_branch:          devel
git_last_commit:     5430b4a
git_last_commit_date:
                     2023-04-25
Date/Publication:    2023-05-23
Author:              John D. Storey [aut,
                     cre], Andrew J. Bass
                     [aut], Alan Dabney
                     [aut], David
                     Robinson [aut],
                     Gregory Warnes [ctb]
Built:               R 4.3.0; ;
                     2023-05-23 12:35:58
                     UTC; unix

Index:

empPvals                Calculate p-values from a set of observed test
                        statistics and simulated null test statistics
hedenfalk               P-values and test-statistics from the Hedenfalk
                        et al. (2001) gene expression dataset
hist.qvalue             Histogram of p-values
lfdr                    Estimate local False Discovery Rate (FDR)
pi0est                  Proportion of true null p-values
plot.qvalue             Plotting function for q-value object
qvalue                  Estimate the q-values for a given set of
                        p-values
summary.qvalue          Display q-value object
write.qvalue            Write results to file

Further information is available in the
following vignettes in directory
&#x27;/tmp/RtmpM6qLgv/Rinst1429e68a70ca6/qvalue/doc&#x27;:

qvalue: qvalue Package (source, pdf)

Warning: &#x27;mode(width)&#x27; differs between new and previous
	 ==&gt; NOT changing &#x27;width&#x27;
Warning: &#x27;mode(width)&#x27; differs between new and previous
	 ==&gt; NOT changing &#x27;width&#x27;
Warning: &#x27;mode(width)&#x27; differs between new and previous
	 ==&gt; NOT changing &#x27;width&#x27;
Warning: &#x27;mode(width)&#x27; differs between new and previous
	 ==&gt; NOT changing &#x27;width&#x27;
Error: processing vignette &#x27;qvalue.Rnw&#x27; failed with diagnostics:
unable to run pdflatex on &#x27;qvalue.tex&#x27;
LaTeX errors:
! LaTeX Error: File `framed.sty&#x27; not found.

Type X to quit or &lt;RETURN&gt; to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
&lt;read *&gt; 
         
l.27 \makeatletter
                  ^^M
!  ==&gt; Fatal error occurred, no output PDF file produced!
--- failed re-building ‘qvalue.Rnw’

SUMMARY: processing the following file failed:
  ‘qvalue.Rnw’

Error: Vignette re-building failed.
Execution halted
</PRE>
</DIV></BODY>
</HTML>
