<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - BUILD results for VariantFiltering on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><B>[<A href="..#V">V</A>]</B></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-24 01:46:23 -0000 (Wed, 24 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4131</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">BUILD results for VariantFiltering on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the VariantFiltering package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/VariantFiltering.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2149</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc selected">BUILD</TD><TD class="STAGE checksrc">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">VariantFiltering</A>&nbsp;1.37.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/VariantFiltering">landing page</A>)</SPAN><BR>Robert Castelo<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-22&nbsp;17:00:18&nbsp;-0000&nbsp;(Mon,&nbsp;22&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/VariantFiltering</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">e4f3be4</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:31:35&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc selected"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc"><SPAN class="glyph skipped">skipped</SPAN></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: VariantFiltering</TD></TR>
<TR><TD><B>Version</B>: 1.37.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data VariantFiltering</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-23 14:42:54 -0000 (Tue, 23 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-23 14:47:51 -0000 (Tue, 23 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 296.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>PackageFile</B>: None</TD></TR>
<TR><TD><B>PackageFileSize</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data VariantFiltering
###
##############################################################################
##############################################################################


* checking for file ‘VariantFiltering/DESCRIPTION’ ... OK
* preparing ‘VariantFiltering’:
* checking DESCRIPTION meta-information ... OK
* cleaning src
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘usingVariantFiltering.Rnw’ using Sweave
Loading required package: BiocGenerics

Attaching package: ‘BiocGenerics’

The following objects are masked from ‘package:stats’:

    IQR, mad, sd, var, xtabs

The following objects are masked from ‘package:base’:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: VariantAnnotation
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: ‘MatrixGenerics’

The following objects are masked from ‘package:matrixStats’:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: GenomeInfoDb
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: ‘S4Vectors’

The following object is masked from ‘package:utils’:

    findMatches

The following objects are masked from ‘package:base’:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: SummarizedExperiment
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: ‘Biobase’

The following object is masked from ‘package:MatrixGenerics’:

    rowMedians

The following objects are masked from ‘package:matrixStats’:

    anyMissing, rowMedians

Loading required package: Rsamtools
Loading required package: Biostrings
Loading required package: XVector

Attaching package: ‘Biostrings’

The following object is masked from ‘package:base’:

    strsplit


Attaching package: ‘VariantAnnotation’

The following object is masked from ‘package:base’:

    tabulate

Loading BSgenome annotation package BSgenome.Hsapiens.1000genomes.hs37d5
Loading OrgDb annotation package org.Hs.eg.db
Loading TxDb annotation package TxDb.Hsapiens.UCSC.hg19.knownGene
Loading SNPlocs annotation package SNPlocs.Hsapiens.dbSNP144.GRCh37
Warning: replacing previous import &#x27;utils::findMatches&#x27; by &#x27;S4Vectors::findMatches&#x27; when loading &#x27;SNPlocs.Hsapiens.dbSNP144.GRCh37&#x27;
Loading annotation package MafDb.1Kgenomes.phase1.hs37d5
Warning: replacing previous import &#x27;utils::findMatches&#x27; by &#x27;S4Vectors::findMatches&#x27; when loading &#x27;MafDb.1Kgenomes.phase1.hs37d5&#x27;
Loading annotation package PolyPhen.Hsapiens.dbSNP131
Loading annotation package SIFT.Hsapiens.dbSNP137
Loading annotation package phastCons100way.UCSC.hg19
Warning: replacing previous import &#x27;utils::findMatches&#x27; by &#x27;S4Vectors::findMatches&#x27; when loading &#x27;phastCons100way.UCSC.hg19&#x27;
Fetching annotation object humanGenesPhylostrata
Assuming the genome build of the input variants is hs37d5.
Switching to the UCSC chromosome-name style from the transcript-centric annotation package.
Chromosome chrM has different lengths between the input VCF and the input TxDb pakage. This chromosome will be discarded from further analysis
Assumming hs37d5 and hg19 represent the same genome build.
Discarding scaffold sequences.
Annotating variant type (SNV, Insertion, Deletion, MNV, Delins)
Annotating dbSNP identifiers with SNPlocs.Hsapiens.dbSNP144.GRCh37
Annotating location with VariantAnnotation::locateVariants()
Warning in valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 597 out-of-bound ranges located on sequences
  29194, 29195, 29196, 72038, 72039, 72040, 72041, 72044, 72220, 72222,
  72223, 72224, 71343, 71344, 73323, 73324, 73326, 72882, 72883, 74544,
  73782, 73852, 74055, 74056, 74057, 74058, 74171, 75259, 76673, 76674,
  76675, 76676, 76677, 76679, and 76680. Note that ranges located on a
  sequence whose length is unknown (NA) or on a circular sequence are not
  considered out-of-bound (use seqlengths() and isCircular() to get the
  lengths and circularity flags of the underlying sequences). You can use
  trim() to trim these ranges. See ?`trim,GenomicRanges-method` for more
  information.
Annotating coding variants VariantAnnotation::predictCoding()
Warning in (function (seqlevels, genome, new_style)  :
  cannot switch some hg19&#x27;s seqlevels from UCSC to NCBI style
Warning in annotationEngine(variants, param, annotationCache, BPPARAM = BPPARAM) :
  Assumming GRCh37.p13 and hs37d5 represent the same genome build.
Warning in valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 597 out-of-bound ranges located on sequences
  29194, 29195, 29196, 72038, 72039, 72040, 72041, 72044, 72220, 72222,
  72223, 72224, 71343, 71344, 73323, 73324, 73326, 72882, 72883, 74544,
  73782, 73852, 74055, 74056, 74057, 74058, 74171, 75259, 76673, 76674,
  76675, 76676, 76677, 76679, and 76680. Note that ranges located on a
  sequence whose length is unknown (NA) or on a circular sequence are not
  considered out-of-bound (use seqlengths() and isCircular() to get the
  lengths and circularity flags of the underlying sequences). You can use
  trim() to trim these ranges. See ?`trim,GenomicRanges-method` for more
  information.
Annotating codon usage frequencies in coding synonymous variants
Annotating with MafDb.1Kgenomes.phase1.hs37d5
Warning in .local(annObj, ...) :
  Assumming GRCh37.p13 and hs37d5 represent the same genome build between variants and the GScores package, respectively.
no MafDb scores for population AF in sequence Y from GScores object x (MafDb.1Kgenomes.phase1.hs37d5).
no MafDb scores for population AFR_AF in sequence Y from GScores object x (MafDb.1Kgenomes.phase1.hs37d5).
no MafDb scores for population AMR_AF in sequence Y from GScores object x (MafDb.1Kgenomes.phase1.hs37d5).
no MafDb scores for population ASN_AF in sequence Y from GScores object x (MafDb.1Kgenomes.phase1.hs37d5).
no MafDb scores for population EUR_AF in sequence Y from GScores object x (MafDb.1Kgenomes.phase1.hs37d5).
Warning in .merge_two_Seqinfo_objects(x, y) :
  The 2 combined objects have no sequence levels in common. (Use
  suppressWarnings() to suppress this warning.)
Warning in .merge_two_Seqinfo_objects(x, y) :
  The 2 combined objects have no sequence levels in common. (Use
  suppressWarnings() to suppress this warning.)
Warning in .merge_two_Seqinfo_objects(x, y) :
  The 2 combined objects have no sequence levels in common. (Use
  suppressWarnings() to suppress this warning.)
Annotating with PolyPhen.Hsapiens.dbSNP131
Warning: 349 keys not found in PolyPhen database: rs760352870 rs771898125 ... rs1543781 rs112153384
Annotating with SIFT.Hsapiens.dbSNP137
Warning: 332 keys not found in PROVEAN database: 760352870 771898125 ... 1543781 112153384
Annotating with phastCons100way.UCSC.hg19
Warning in .local(annObj, ...) :
  Assumming hg19 and Genome Reference Consortium GRCh37 represent the same genome build between variants and the GScores package, respectively.
no phastCons100way.UCSC.hg19 scores for nonSNRs in sequence chr20 from GScores object x.
no phastCons100way.UCSC.hg19 scores for nonSNRs in sequence chr21 from GScores object x.
no phastCons100way.UCSC.hg19 scores for nonSNRs in sequence chr22 from GScores object x.
no phastCons100way.UCSC.hg19 scores for nonSNRs in sequence chrX from GScores object x.
Annotating with humanGenesPhylostrata
1000 variants processed
Warning in (function (seqlevels, genome, new_style)  :
  cannot switch some hg19&#x27;s seqlevels from UCSC to NCBI style
Error: processing vignette &#x27;usingVariantFiltering.Rnw&#x27; failed with diagnostics:
unable to run pdflatex on &#x27;usingVariantFiltering.tex&#x27;
LaTeX errors:
! LaTeX Error: File `framed.sty&#x27; not found.

Type X to quit or &lt;RETURN&gt; to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
&lt;read *&gt; 
         
l.137 \definecolor
                  {shadecolor}{RGB}{240,240,240}^^M
!  ==&gt; Fatal error occurred, no output PDF file produced!
--- failed re-building &#x27;usingVariantFiltering.Rnw&#x27;

SUMMARY: processing the following file failed:
  &#x27;usingVariantFiltering.Rnw&#x27;

Error: Vignette re-building failed.
Execution halted
</PRE>
</DIV></BODY>
</HTML>
