<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for ensembldb on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><B>[<A href="..#E">E</A>]</B></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-06 11:00:31 -0000 (Tue, 06 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for ensembldb on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ensembldb package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ensembldb.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>626</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ensembldb</A>&nbsp;2.25.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/ensembldb">landing page</A>)</SPAN><BR>Johannes Rainer<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-05&nbsp;06:35:06&nbsp;-0000&nbsp;(Mon,&nbsp;05&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ensembldb</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">62012ae</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:38:28&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ensembldb</TD></TR>
<TR><TD><B>Version</B>: 2.25.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:ensembldb.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings ensembldb_2.25.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-05 21:47:59 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-05 22:06:20 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1100.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ensembldb.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:ensembldb.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings ensembldb_2.25.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/ensembldb.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘ensembldb/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘ensembldb’ version ‘2.25.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ensembldb’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                user system elapsed
Filter-classes 2.247  1.001   5.123
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
   2.   └─ensembldb:::.getReadMysqlTable(...)
   3.     └─utils::download.file(...)
  ── Error (&#x27;test_functions-create-EnsDb.R:258:5&#x27;): .read_lines_compressed works ──
  Error in `file(x, open = &quot;rb&quot;)`: cannot open the connection to &#x27;ftp://ftp.ensembl.org/pub/release-102/gtf/homo_sapiens/Homo_sapiens.GRCh38.102.gtf.gz&#x27;
  Backtrace:
      ▆
   1. └─ensembldb:::.read_lines_compressed(fl, n = 2) at test_functions-create-EnsDb.R:258:4
   2.   ├─base::gzcon(file(x, open = &quot;rb&quot;))
   3.   └─base::file(x, open = &quot;rb&quot;)
  
  [ FAIL 3 | WARN 82 | SKIP 3 | PASS 1471 ]
  Error: Test failures
  In addition: Warning message:
  call dbDisconnect() when finished working with a connection 
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘MySQL-backend.Rmd’ using ‘UTF-8’... OK
  ‘coordinate-mapping-use-cases.Rmd’ using ‘UTF-8’... OK
  ‘coordinate-mapping.Rmd’ using ‘UTF-8’... OK
  ‘ensembldb.Rmd’ using ‘UTF-8’... OK
  ‘proteins.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/ensembldb.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ensembldb.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL ensembldb
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘ensembldb’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ensembldb)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">ensembldb.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(ensembldb)
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicFeatures
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: AnnotationFilter

Attaching package: &#x27;AnnotationFilter&#x27;

The following object is masked from &#x27;package:testthat&#x27;:

    not


Attaching package: &#x27;ensembldb&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

&gt; library(EnsDb.Hsapiens.v86)
&gt; edb &lt;- EnsDb.Hsapiens.v86
&gt; 
&gt; test_check(&quot;ensembldb&quot;)
Creating package in /tmp/RtmpVuEytu/EnsDb.Hsapiens.v75 
Error in readLines(curl(my_url)) : 
  Timeout was reached: [ftp.ensemblgenomes.org] Connection timeout after 10000 ms

Comparing chromosome data:
 Sequence names: (180) common, (0) only in x, (0) only in y.
 Sequence lengths: (180) identical, (0) different.
Done. Result: OK

Comparing gene data:
 gene IDs: (4045) common, (0) only in x, (0) only in y.
 Sequence names: (4045) identical, (0) different.
 Gene start coordinates: (4045) identical, (0) different.
 Gene end coordinates: (4045) identical, (0) different.
 Gene strand: (4045) identical, (0) different.
 Gene names: (132) identical, (3913) different.
 Entrezgene IDs: (4045) identical, (0) different.
 Gene biotypes: (4045) identical, (0) different.
Done. Result: WARN

Comparing transcript data:
 transcript IDs: (4045) common, (0) only in x, (0) only in y.
 Transcript start coordinates: (4045) identical, (0) different.
 Transcript end coordinates: (4045) identical, (0) different.
 Transcript biotypes: (4045) identical, (0) different.
 Common transcripts with defined CDS: (3996) common, (0) only in x, (0) only in y.
 CDS start coordinates: (3996) identical, (0) different.
 CDS end coordinates: (3996) identical, (0) different.
 Associated gene IDs: (4045) identical, (0) different.
Done. Result: OK

Comparing exon data:
 exon IDs: (4045) common, (0) only in x, (0) only in y.
 Exon start coordinates: (4045) identical, (0) different.
 Exon end coordinates: (4045) identical, (0) different.
 Exon index in transcript models: (4045) identical, (0) different.
Done. Result: OK

Comparing metadata:
 Ensembl versions match.
 Genome builds match.
 All differences: &lt;name&gt;: &lt;value x&gt; != &lt;value y&gt;
  -  Creation time : Mon Jun  5 21:56:29 2023  !=  Mon Jun  5 21:56:01 2023 
  -  source_file : Devosia_geojensis.ASM96941v1.32.gtf.gz  !=  Devosia_geojensis.ASM96941v1.32.gff3.gz 
Done. Result: NOTE

Comparing chromosome data:
 Sequence names: (180) common, (0) only in x, (0) only in y.
 Sequence lengths: (180) identical, (0) different.
Done. Result: OK

Comparing gene data:
 gene IDs: (4045) common, (0) only in x, (0) only in y.
 Sequence names: (4045) identical, (0) different.
 Gene start coordinates: (4045) identical, (0) different.
 Gene end coordinates: (4045) identical, (0) different.
 Gene strand: (4045) identical, (0) different.
 Gene names: (132) identical, (3913) different.
 Entrezgene IDs: (4045) identical, (0) different.
 Gene biotypes: (4045) identical, (0) different.
Done. Result: WARN

Comparing transcript data:
 transcript IDs: (4045) common, (0) only in x, (0) only in y.
 Transcript start coordinates: (4045) identical, (0) different.
 Transcript end coordinates: (4045) identical, (0) different.
 Transcript biotypes: (4045) identical, (0) different.
 Common transcripts with defined CDS: (3996) common, (0) only in x, (0) only in y.
 CDS start coordinates: (3996) identical, (0) different.
 CDS end coordinates: (3996) identical, (0) different.
 Associated gene IDs: (4045) identical, (0) different.
Done. Result: OK

Comparing exon data:
 exon IDs: (4045) common, (0) only in x, (0) only in y.
 Exon start coordinates: (4045) identical, (0) different.
 Exon end coordinates: (4045) identical, (0) different.
 Exon index in transcript models: (4045) identical, (0) different.
Done. Result: OK

Comparing protein data:
 protein IDs: (103725) common, (0) only in x, (0) only in y.
 Transcript IDs: (103725) identical, (0) different.
 Protein sequence: (103725) identical, (0) different.
Done. Result: OK
[ FAIL 3 | WARN 82 | SKIP 3 | PASS 1471 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• empty test (3)

══ Failed tests ════════════════════════════════════════════════════════════════
── Error (&#x27;test_functions-create-EnsDb.R:183:9&#x27;): getEnsemblMysqlUrl works ─────
Error in `readLines(curl(my_url))`: Timeout was reached: [ftp.ensembl.org] Connection timeout after 10001 ms
Backtrace:
    ▆
 1. └─ensembldb:::.getEnsemblMysqlUrl(...) at test_functions-create-EnsDb.R:183:8
 2.   └─base::readLines(curl(my_url))
── Error (&#x27;test_functions-create-EnsDb.R:234:9&#x27;): getSeqlengthsFromMysqlFolder works ──
Error in `download.file(url = paste0(base_url, &quot;/&quot;, file_name), destfile = tmp_file, 
    quiet = TRUE)`: cannot open URL &#x27;ftp://ftp.ensembl.org/pub/release-86/mysql/homo_sapiens_core_86_38/coord_system.txt.gz&#x27;
Backtrace:
    ▆
 1. └─ensembldb:::.getSeqlengthsFromMysqlFolder(...) at test_functions-create-EnsDb.R:234:8
 2.   └─ensembldb:::.getReadMysqlTable(...)
 3.     └─utils::download.file(...)
── Error (&#x27;test_functions-create-EnsDb.R:258:5&#x27;): .read_lines_compressed works ──
Error in `file(x, open = &quot;rb&quot;)`: cannot open the connection to &#x27;ftp://ftp.ensembl.org/pub/release-102/gtf/homo_sapiens/Homo_sapiens.GRCh38.102.gtf.gz&#x27;
Backtrace:
    ▆
 1. └─ensembldb:::.read_lines_compressed(fl, n = 2) at test_functions-create-EnsDb.R:258:4
 2.   ├─base::gzcon(file(x, open = &quot;rb&quot;))
 3.   └─base::file(x, open = &quot;rb&quot;)

[ FAIL 3 | WARN 82 | SKIP 3 | PASS 1471 ]
Error: Test failures
In addition: Warning message:
call dbDisconnect() when finished working with a connection 
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ensembldb.Rcheck/ensembldb-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>EnsDb-AnnotationDbi</TD><TD style="text-align: right;">2.475</TD><TD style="text-align: right;">0.475</TD><TD style="text-align: right;">4.271</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-class</TD><TD style="text-align: right;">1.817</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">1.954</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-exonsBy</TD><TD style="text-align: right;">2.300</TD><TD style="text-align: right;">0.421</TD><TD style="text-align: right;">4.305</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-lengths</TD><TD style="text-align: right;">1.230</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">1.309</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-seqlevels</TD><TD style="text-align: right;">0.058</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-sequences</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-utils</TD><TD style="text-align: right;">0.324</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.325</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb</TD><TD style="text-align: right;">0.872</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">0.949</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Filter-classes</TD><TD style="text-align: right;">2.247</TD><TD style="text-align: right;">1.001</TD><TD style="text-align: right;">5.123</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ProteinFunctionality</TD><TD style="text-align: right;">0.079</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cdsToTranscript</TD><TD style="text-align: right;">1.949</TD><TD style="text-align: right;">0.057</TD><TD style="text-align: right;">2.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertFilter</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomeToProtein</TD><TD style="text-align: right;">2.312</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">2.341</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomeToTranscript</TD><TD style="text-align: right;">1.968</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">2.037</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>global-filters</TD><TD style="text-align: right;">0.142</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.143</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hasProteinData-EnsDb-method</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>listEnsDbs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeEnsemblDbPackage</TD><TD style="text-align: right;">0.890</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.924</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>proteinToGenome</TD><TD style="text-align: right;">2.677</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">2.753</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>proteinToTranscript</TD><TD style="text-align: right;">1.124</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">1.202</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptToCds</TD><TD style="text-align: right;">0.739</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.742</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptToGenome</TD><TD style="text-align: right;">1.421</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">1.456</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptToProtein</TD><TD style="text-align: right;">2.458</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">2.488</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>useMySQL-EnsDb-method</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
