<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for ReUseData on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-05 06:03:07 -0000 (Mon, 05 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for ReUseData on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ReUseData package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ReUseData.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1667</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ReUseData</A>&nbsp;1.1.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/ReUseData">landing page</A>)</SPAN><BR>Qian Liu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-03&nbsp;10:56:11&nbsp;-0000&nbsp;(Sat,&nbsp;03&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ReUseData</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">3a1df6c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:42:19&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ReUseData</TD></TR>
<TR><TD><B>Version</B>: 1.1.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:ReUseData.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings ReUseData_1.1.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-04 22:35:31 -0000 (Sun, 04 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-04 22:38:51 -0000 (Sun, 04 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 200.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ReUseData.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:ReUseData.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings ReUseData_1.1.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory â€˜/home/biocbuild/bbs-3.18-bioc/meat/ReUseData.Rcheckâ€™
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file â€˜ReUseData/DESCRIPTIONâ€™ ... OK
* this is package â€˜ReUseDataâ€™ version â€˜1.1.0â€™
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section â€˜Package
structureâ€™ in the â€˜Writing R Extensionsâ€™ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package â€˜ReUseDataâ€™ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking â€˜buildâ€™ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in â€˜vignettesâ€™ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                user system elapsed
dataUpdate    15.969  1.105  17.681
getCloudData   9.854  0.450  12.523
dataHub-class  8.543  0.891   9.457
getData        4.844  0.235   5.085
* checking for unstated dependencies in â€˜testsâ€™ ... OK
* checking tests ...
  Running â€˜testthat.Râ€™
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in â€˜inst/docâ€™ ... OK
* checking running R code from vignettes ...
  â€˜ReUseData_data.Rmdâ€™ using â€˜UTF-8â€™... OK
  â€˜ReUseData_quickStart.Rmdâ€™ using â€˜UTF-8â€™... OK
  â€˜ReUseData_recipe.Rmdâ€™ using â€˜UTF-8â€™... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building â€˜ReUseData_data.Rmdâ€™ using rmarkdown
Error: processing vignette &#x27;ReUseData_data.Rmd&#x27; failed with diagnostics:
there is no package called â€˜BiocStyleâ€™
--- failed re-building â€˜ReUseData_data.Rmdâ€™

--- re-building â€˜ReUseData_quickStart.Rmdâ€™ using rmarkdown
Error: processing vignette &#x27;ReUseData_quickStart.Rmd&#x27; failed with diagnostics:
there is no package called â€˜BiocStyleâ€™
--- failed re-building â€˜ReUseData_quickStart.Rmdâ€™

--- re-building â€˜ReUseData_recipe.Rmdâ€™ using rmarkdown
Error: processing vignette &#x27;ReUseData_recipe.Rmd&#x27; failed with diagnostics:
there is no package called â€˜BiocStyleâ€™
--- failed re-building â€˜ReUseData_recipe.Rmdâ€™

SUMMARY: processing the following files failed:
  â€˜ReUseData_data.Rmdâ€™ â€˜ReUseData_quickStart.Rmdâ€™
  â€˜ReUseData_recipe.Rmdâ€™

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 NOTE
See
  â€˜/home/biocbuild/bbs-3.18-bioc/meat/ReUseData.Rcheck/00check.logâ€™
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ReUseData.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL ReUseData
###
##############################################################################
##############################################################################


* installing to library â€˜/home/biocbuild/R/R-4.3.0/site-libraryâ€™
* installing *source* package â€˜ReUseDataâ€™ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ReUseData)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">ReUseData.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(ReUseData)
&gt; 
&gt; test_check(&quot;ReUseData&quot;)
Note: you need to assign a name for the recipe: rcpName &lt;- recipeLoad(&#x27;xx&#x27;)
Data recipe loaded!
Use inputs() to check required input parameters before evaluation.
Check here: https://rcwl.org/dataRecipes/echo_out.html
for user instructions (e.g., eligible input values, data source, etc.)

}[1;30mINFO[0m Final process status is success

Updating data record...
outfile.txt added

Updating data record...
22a13559f646bd_GRCh38.primary_assembly.genome.fa.1.bt2 added
22a1356b6e272e_GRCh38.primary_assembly.genome.fa.2.bt2 added
22a1355a58f7f9_GRCh38.primary_assembly.genome.fa.3.bt2 added
22a1352b2bc3e7_GRCh38.primary_assembly.genome.fa.4.bt2 added
22a1356d43ad4c_GRCh38.primary_assembly.genome.fa.rev.1.bt2 added
22a13536af3838_GRCh38.primary_assembly.genome.fa.rev.2.bt2 added
22a135468e815_outfile.txt added
22a1351120ca34_GRCh37_to_GRCh38.chain added
22a1357d77c762_GRCh37_to_NCBI34.chain added
22a13570c6e581_GRCh37_to_NCBI35.chain added
22a1351d855d91_GRCh37_to_NCBI36.chain added
22a135312510c5_GRCh38_to_GRCh37.chain added
22a1354b0650d_GRCh38_to_NCBI34.chain added
22a13543b604ca_GRCh38_to_NCBI35.chain added
22a135196b78db_GRCh38_to_NCBI36.chain added
22a1356fa18717_NCBI34_to_GRCh37.chain added
22a1354dc4ff27_NCBI34_to_GRCh38.chain added
22a13522d25899_NCBI35_to_GRCh37.chain added
22a135c1f63e0_NCBI35_to_GRCh38.chain added
22a13572c66ea3_NCBI36_to_GRCh37.chain added
22a135547a8bcb_NCBI36_to_GRCh38.chain added
22a1355a28cb71_GRCm38_to_NCBIM36.chain added
22a13516054cde_GRCm38_to_NCBIM37.chain added
22a135681b397c_NCBIM36_to_GRCm38.chain added
22a1351f6838ed_NCBIM37_to_GRCm38.chain added
22a1352f7b99a7_1000G_omni2.5.b37.vcf.gz added
22a13513ebaf7_1000G_omni2.5.b37.vcf.gz.tbi added
22a13527bf47b4_Mills_and_1000G_gold_standard.indels.b37.vcf.gz added
22a1354dab1ca2_Mills_and_1000G_gold_standard.indels.b37.vcf.gz.tbi added
22a135215a70e2_1000G_omni2.5.hg38.vcf.gz added
22a1355b8901d1_1000G_omni2.5.hg38.vcf.gz.tbi added
22a13527a16360_Mills_and_1000G_gold_standard.indels.hg38.vcf.gz added
22a135cc89810_Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi added
22a13535e1f9cb_af-only-gnomad.raw.sites.vcf added
22a13552cd2747_af-only-gnomad.raw.sites.vcf.idx added
22a1357a0c455d_Mutect2-exome-panel.vcf.idx added
22a1356c913203_Mutect2-WGS-panel-b37.vcf added
22a13557360f5c_Mutect2-WGS-panel-b37.vcf.idx added
22a135b2d0f91_small_exac_common_3.vcf added
22a1356a08f966_small_exac_common_3.vcf.idx added
22a13547fcf4dd_1000g_pon.hg38.vcf.gz added
22a13528b26d22_1000g_pon.hg38.vcf.gz.tbi added
22a1351b2e0a2b_af-only-gnomad.hg38.vcf.gz added
22a1354cad59eb_af-only-gnomad.hg38.vcf.gz.tbi added
22a1356c6871ed_small_exac_common_3.hg38.vcf.gz added
22a13534998306_small_exac_common_3.hg38.vcf.gz.tbi added
22a1353c4ee102_gencode.v41.annotation.gtf added
22a1353a2d7114_gencode.v42.annotation.gtf added
22a135576bdb9f_gencode.vM30.annotation.gtf added
22a135486e44e2_gencode.vM31.annotation.gtf added
22a1352cf3dfb7_gencode.v41.transcripts.fa added
22a1352be6676a_gencode.v41.transcripts.fa.fai added
22a13522971053_gencode.v42.transcripts.fa added
22a13542f92c95_gencode.v42.transcripts.fa.fai added
22a1351401a0e6_gencode.vM30.pc_transcripts.fa added
22a13541ff4940_gencode.vM30.pc_transcripts.fa.fai added
22a1357274c63c_gencode.vM31.pc_transcripts.fa added
22a13515405bdd_gencode.vM31.pc_transcripts.fa.fai added
22a13569be90f4_GRCh38.primary_assembly.genome.fa.1.ht2 added
22a135401fe2df_GRCh38.primary_assembly.genome.fa.2.ht2 added
22a135369accc0_GRCh38.primary_assembly.genome.fa.3.ht2 added
22a135454792c6_GRCh38.primary_assembly.genome.fa.4.ht2 added
22a13567c1463f_GRCh38.primary_assembly.genome.fa.5.ht2 added
22a135436364d0_GRCh38.primary_assembly.genome.fa.6.ht2 added
22a1357b298c91_GRCh38.primary_assembly.genome.fa.7.ht2 added
22a1353a8e6d86_GRCh38.primary_assembly.genome.fa.8.ht2 added
22a1353d6faa2d_GRCh38_full_analysis_set_plus_decoy_hla.fa.fai added
22a13567babe94_GRCh38_full_analysis_set_plus_decoy_hla.fa.amb added
22a13511c47ce2_GRCh38_full_analysis_set_plus_decoy_hla.fa.ann added
22a135489cb9bf_GRCh38_full_analysis_set_plus_decoy_hla.fa.bwt added
22a13551c3b7fa_GRCh38_full_analysis_set_plus_decoy_hla.fa.pac added
22a13559c171c0_GRCh38_full_analysis_set_plus_decoy_hla.fa.sa added
22a135714f26e1_GRCh38_full_analysis_set_plus_decoy_hla.fa added
22a1356cf1c225_GRCh38.primary_assembly.genome.fa.fai added
22a135266ecbab_GRCh38.primary_assembly.genome.fa.amb added
22a1355db798ce_GRCh38.primary_assembly.genome.fa.ann added
22a135218b452b_GRCh38.primary_assembly.genome.fa.bwt added
22a13562bdacad_GRCh38.primary_assembly.genome.fa.pac added
22a13517e509e2_GRCh38.primary_assembly.genome.fa.sa added
22a13578f720ca_GRCh38.primary_assembly.genome.fa added
22a1352b2bf18f_hs37d5.fa.fai added
22a13544d8e999_hs37d5.fa.amb added
22a13524dd8834_hs37d5.fa.ann added
22a1354dc301e2_hs37d5.fa.bwt added
22a1357d2162e_hs37d5.fa.pac added
22a13538df291a_hs37d5.fa.sa added
22a135fc24b22_hs37d5.fa added
22a1357a46dc6b_complete_ref_lens.bin added
22a1354e1f84f8_ctable.bin added
22a1357980dc17_ctg_offsets.bin added
22a1353a66bf4a_duplicate_clusters.tsv added
22a1354ba51b8_info.json added
22a1353ec86edd_mphf.bin added
22a13522280589_pos.bin added
22a135481db688_pre_indexing.log added
22a13539f1fb6e_rank.bin added
22a1355cb6730f_ref_indexing.log added
22a13558d60b6_refAccumLengths.bin added
22a13521acba02_reflengths.bin added
22a1356e7aeff1_refseq.bin added
22a1354e2a1a75_seq.bin added
22a135737071fd_versionInfo.json added
22a135483c61b1_salmon_index added
22a1353f794156_chrLength.txt added
22a13560623422_chrName.txt added
22a1356eab2d5c_chrNameLength.txt added
22a1351d30da25_chrStart.txt added
22a1351ed794e_exonGeTrInfo.tab added
22a1355168da09_exonInfo.tab added
22a1353515e407_geneInfo.tab added
22a1357ae49a18_Genome added
22a1357c94cb98_genomeParameters.txt added
22a13579eecda1_Log.out added
22a1351fc2224d_SA added
22a1354a57cd7a_SAindex added
22a1351c0e3cf_sjdbInfo.txt added
22a13558a14b67_sjdbList.fromGTF.out.tab added
22a1355a1a189d_sjdbList.out.tab added
22a1357c07c03a_transcriptInfo.tab added
22a13526c0d05f_GRCh38.GENCODE.v42_100 added
22a135539af4b4_knownGene_hg38.sql added
22a135366e7f84_knownGene_hg38.txt added
22a1352b7b2217_refGene_hg38.sql added
22a13512636391_refGene_hg38.txt added
22a1355896850d_knownGene_mm39.sql added
22a1357398d8a0_knownGene_mm39.txt added
22a1354c555eff_refGene_mm39.sql added
22a135354cf81c_refGene_mm39.txt added
trying URL &#x27;https://storage.googleapis.com/reusedata/echo/outfile.txt&#x27;
Content type &#x27;text/plain&#x27; length 30 bytes
==================================================
downloaded 30 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.yml&#x27;
Content type &#x27;application/x-yaml&#x27; length 151 bytes
==================================================
downloaded 151 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.cwl&#x27;
Content type &#x27;application/octet-stream&#x27; length 1152 bytes
==================================================
downloaded 1152 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.md5&#x27;
Content type &#x27;application/octet-stream&#x27; length 45 bytes
==================================================
downloaded 45 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.sh&#x27;
Content type &#x27;application/x-shellscript&#x27; length 68 bytes
==================================================
downloaded 68 bytes

Data is downloaded: 
/tmp/RtmpA7QJDi/test_gcpData/outfile.txt
}[1;30mINFO[0m Final process status is success
NOTE: existing caches will be removed and regenerated!
Updating recipes...
STAR_index.R added
bowtie2_index.R added
echo_out.R added
ensembl_liftover.R added
gcp_broad_gatk_hg19.R added
gcp_broad_gatk_hg38.R added
gcp_gatk_mutect2_b37.R added
gcp_gatk_mutect2_hg38.R added
gencode_annotation.R added
gencode_genome_grch38.R added
gencode_transcripts.R added
hisat2_index.R added
reference_genome.R added
salmon_index.R added
ucsc_database.R added

Data recipe loaded!
Use inputs(ensembl_liftover) to check required input parameters before evaluation.
Check here: https://rcwl.org/dataRecipes/ensembl_liftover.html
for user instructions (e.g., eligible input values, data source, etc.)

[ FAIL 0 | WARN 0 | SKIP 0 | PASS 44 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 31.512   2.157  36.263 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ReUseData.Rcheck/ReUseData-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>dataHub-class</TD><TD style="text-align: right;">8.543</TD><TD style="text-align: right;">0.891</TD><TD style="text-align: right;">9.457</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dataSearch</TD><TD style="text-align: right;">1.371</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">1.422</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dataUpdate</TD><TD style="text-align: right;">15.969</TD><TD style="text-align: right;"> 1.105</TD><TD style="text-align: right;">17.681</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCloudData</TD><TD style="text-align: right;"> 9.854</TD><TD style="text-align: right;"> 0.450</TD><TD style="text-align: right;">12.523</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getData</TD><TD style="text-align: right;">4.844</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">5.085</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meta_data</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeHub-class</TD><TD style="text-align: right;">0.234</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.254</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeLoad</TD><TD style="text-align: right;">1.491</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">1.561</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeMake</TD><TD style="text-align: right;">4.663</TD><TD style="text-align: right;">0.211</TD><TD style="text-align: right;">4.878</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeSearch</TD><TD style="text-align: right;">0.697</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">0.747</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeUpdate</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
