<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for chimeraviz on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><B>[<A href="..#C">C</A>]</B></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-06 11:00:28 -0000 (Tue, 06 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for chimeraviz on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the chimeraviz package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/chimeraviz.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>318</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">chimeraviz</A>&nbsp;1.27.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/chimeraviz">landing page</A>)</SPAN><BR>Stian Lågstad<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-05&nbsp;06:35:06&nbsp;-0000&nbsp;(Mon,&nbsp;05&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/chimeraviz</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">44b1bf7</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-05-28&nbsp;20:24:23&nbsp;-0000&nbsp;(Sun,&nbsp;28&nbsp;May&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: chimeraviz</TD></TR>
<TR><TD><B>Version</B>: 1.27.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:chimeraviz.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings chimeraviz_1.27.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-05 19:16:48 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-05 19:34:11 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1042.5 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: chimeraviz.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:chimeraviz.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings chimeraviz_1.27.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/chimeraviz.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘chimeraviz/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘chimeraviz’ version ‘1.27.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;Biostrings&#x27;, &#x27;GenomicRanges&#x27;, &#x27;IRanges&#x27;, &#x27;Gviz&#x27;, &#x27;S4Vectors&#x27;,
  &#x27;ensembldb&#x27;, &#x27;AnnotationFilter&#x27;, &#x27;data.table&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘chimeraviz’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
Namespace in Imports field not imported from: &#x27;magick&#x27;
  All declared Imports should be used.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
plot_fusion_transcript_with_protein_domain: no visible binding for
  global variable &#x27;protein_domain_location&#x27;
Undefined global functions or variables:
  protein_domain_location
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                                             user system elapsed
plot_fusion                                42.296  1.481  51.450
plot_transcripts                           26.017  0.531  26.588
plot_fusion_transcript                     21.137  0.267  21.436
plot_fusion_transcripts_graph              13.584  0.136  13.743
plot_fusion_transcript_with_protein_domain 13.124  0.064  13.207
get_transcripts_ensembl_db                  8.571  0.315   8.904
select_transcript                           8.197  0.032   8.242
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘chimeraviz-vignette.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/chimeraviz.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">chimeraviz.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL chimeraviz
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘chimeraviz’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (chimeraviz)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">chimeraviz.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(chimeraviz)
Loading required package: Biostrings
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: XVector
Loading required package: GenomeInfoDb

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: GenomicRanges
Loading required package: Gviz
Loading required package: grid

Attaching package: &#x27;grid&#x27;

The following object is masked from &#x27;package:Biostrings&#x27;:

    pattern

Loading required package: ensembldb
Loading required package: GenomicFeatures
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: AnnotationFilter

Attaching package: &#x27;AnnotationFilter&#x27;

The following object is masked from &#x27;package:Gviz&#x27;:

    feature

The following object is masked from &#x27;package:testthat&#x27;:

    not


Attaching package: &#x27;ensembldb&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

Loading required package: data.table

Attaching package: &#x27;data.table&#x27;

The following object is masked from &#x27;package:GenomicRanges&#x27;:

    shift

The following object is masked from &#x27;package:IRanges&#x27;:

    shift

The following objects are masked from &#x27;package:S4Vectors&#x27;:

    first, second



&gt; 
&gt; test_check(&quot;chimeraviz&quot;)
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 224 ]
Warning message:
call dbDisconnect() when finished working with a connection 
&gt; 
&gt; proc.time()
   user  system elapsed 
217.138   5.029 228.691 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">chimeraviz.Rcheck/chimeraviz-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>add_fusion_reads_alignment</TD><TD style="text-align: right;">0.943</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.950</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create_fusion_report</TD><TD style="text-align: right;">3.286</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">3.388</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decide_transcript_category</TD><TD style="text-align: right;">0.309</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.311</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>down_shift</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>downstream_partner_gene</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fetch_reads_from_fastq</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fusion_spanning_reads_count</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fusion_split_reads_count</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fusion_to_data_frame</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_ensembl_ids</TD><TD style="text-align: right;">0.605</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.622</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_fusion_by_chromosome</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_fusion_by_gene_name</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_fusion_by_id</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_transcripts_ensembl_db</TD><TD style="text-align: right;">8.571</TD><TD style="text-align: right;">0.315</TD><TD style="text-align: right;">8.904</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_aeron</TD><TD style="text-align: right;">0.640</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.697</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_chimpipe</TD><TD style="text-align: right;">0.169</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.190</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_defuse</TD><TD style="text-align: right;">0.123</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_ericscript</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_fusioncatcher</TD><TD style="text-align: right;">0.123</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_fusionmap</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.134</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_infusion</TD><TD style="text-align: right;">0.134</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.135</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_jaffa</TD><TD style="text-align: right;">0.125</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.130</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_oncofuse</TD><TD style="text-align: right;">0.127</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_prada</TD><TD style="text-align: right;">0.131</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_soapfuse</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_squid</TD><TD style="text-align: right;">0.123</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_starfusion</TD><TD style="text-align: right;">0.164</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.208</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>partner_gene_ensembl_id</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>partner_gene_junction_sequence</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_circle</TD><TD style="text-align: right;">0.741</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.757</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion</TD><TD style="text-align: right;">42.296</TD><TD style="text-align: right;"> 1.481</TD><TD style="text-align: right;">51.450</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion_reads</TD><TD style="text-align: right;">2.700</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">2.775</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion_transcript</TD><TD style="text-align: right;">21.137</TD><TD style="text-align: right;"> 0.267</TD><TD style="text-align: right;">21.436</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion_transcript_with_protein_domain</TD><TD style="text-align: right;">13.124</TD><TD style="text-align: right;"> 0.064</TD><TD style="text-align: right;">13.207</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion_transcripts_graph</TD><TD style="text-align: right;">13.584</TD><TD style="text-align: right;"> 0.136</TD><TD style="text-align: right;">13.743</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_transcripts</TD><TD style="text-align: right;">26.017</TD><TD style="text-align: right;"> 0.531</TD><TD style="text-align: right;">26.588</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>select_transcript</TD><TD style="text-align: right;">8.197</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">8.242</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>split_on_utr_and_add_feature</TD><TD style="text-align: right;">0.589</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.606</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>upstream_partner_gene</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>write_fusion_reference</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
