<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - BUILD results for AneuFinder on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><B>[<A href="..#A">A</A>]</B></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-24 01:46:09 -0000 (Wed, 24 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4131</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">BUILD results for AneuFinder on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the AneuFinder package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/AneuFinder.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>61</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc selected">BUILD</TD><TD class="STAGE checksrc">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">AneuFinder</A>&nbsp;1.29.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/AneuFinder">landing page</A>)</SPAN><BR>Aaron Taudt<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-22&nbsp;17:00:18&nbsp;-0000&nbsp;(Mon,&nbsp;22&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/AneuFinder</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">ec8363a</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:45:33&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc selected"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc"><SPAN class="glyph skipped">skipped</SPAN></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: AneuFinder</TD></TR>
<TR><TD><B>Version</B>: 1.29.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data AneuFinder</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-23 07:46:08 -0000 (Tue, 23 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-23 07:50:37 -0000 (Tue, 23 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 268.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>PackageFile</B>: None</TD></TR>
<TR><TD><B>PackageFileSize</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data AneuFinder
###
##############################################################################
##############################################################################


* checking for file ‘AneuFinder/DESCRIPTION’ ... OK
* preparing ‘AneuFinder’:
* checking DESCRIPTION meta-information ... OK
* cleaning src
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘AneuFinder.Rnw’ using knitr
Aneufinder             package:AneuFinder              R Documentation

_W_r_a_p_p_e_r _f_u_n_c_t_i_o_n _f_o_r _t_h_e &#x27;_A_n_e_u_F_i_n_d_e_r&#x27; _p_a_c_k_a_g_e

_D_e_s_c_r_i_p_t_i_o_n:

     This function is an easy-to-use wrapper to bin the data, find
     copy-number-variations, locate breakpoints, plot genomewide
     heatmaps, distributions, profiles and karyograms.

_U_s_a_g_e:

     Aneufinder(inputfolder, outputfolder, configfile = NULL, numCPU = 1,
       reuse.existing.files = TRUE, binsizes = 1e+06, stepsizes = binsizes,
       variable.width.reference = NULL, reads.per.bin = NULL,
       pairedEndReads = FALSE, assembly = NULL, chromosomes = NULL,
       remove.duplicate.reads = TRUE, min.mapq = 10, blacklist = NULL,
       use.bamsignals = FALSE, reads.store = FALSE, correction.method = NULL,
       GC.BSgenome = NULL, method = c(&quot;edivisive&quot;), strandseq = FALSE,
       R = 10, sig.lvl = 0.1, eps = 0.01, max.time = 60, max.iter = 5000,
       num.trials = 15, states = c(&quot;zero-inflation&quot;, paste0(0:10, &quot;-somy&quot;)),
       confint = NULL, refine.breakpoints = FALSE, hotspot.bandwidth = NULL,
       hotspot.pval = 0.05, cluster.plots = TRUE)
     
_A_r_g_u_m_e_n_t_s:

inputfolder: Folder with either BAM or BED files.

outputfolder: Folder to output the results. If it does not exist it
          will be created.

configfile: A file specifying the parameters of this function (without
          &#x27;inputfolder&#x27;, &#x27;outputfolder&#x27; and &#x27;configfile&#x27;). Having the
          parameters in a file can be handy if many samples with the
          same parameter settings are to be run. If a &#x27;configfile&#x27; is
          specified, it will take priority over the command line
          parameters.

  numCPU: The numbers of CPUs that are used. Should not be more than
          available on your machine.

reuse.existing.files: A logical indicating whether or not existing
          files in &#x27;outputfolder&#x27; should be reused.

binsizes: An integer vector with bin sizes. If more than one value is
          given, output files will be produced for each bin size.

stepsizes: A vector of step sizes the same length as &#x27;binsizes&#x27;. Only
          used for &#x27;method=&quot;HMM&quot;&#x27;.

variable.width.reference: A BAM file that is used as reference to
          produce variable width bins. See &#x27;variableWidthBins&#x27; for
          details.

reads.per.bin: Approximate number of desired reads per bin. The bin
          size will be selected accordingly. Output files are produced
          for each value.

pairedEndReads: Set to &#x27;TRUE&#x27; if you have paired-end reads in your BAM
          files (not implemented for BED files).

assembly: Please see &#x27;getChromInfoFromUCSC&#x27; for available assemblies.
          Only necessary when importing BED files. BAM files are
          handled automatically. Alternatively a data.frame with
          columns &#x27;chromosome&#x27; and &#x27;length&#x27;.

chromosomes: If only a subset of the chromosomes should be imported,
          specify them here.

remove.duplicate.reads: A logical indicating whether or not duplicate
          reads should be removed.

min.mapq: Minimum mapping quality when importing from BAM files. Set
          &#x27;min.mapq=NA&#x27; to keep all reads.

blacklist: A &#x27;GRanges-class&#x27; or a bed(.gz) file with blacklisted
          regions. Reads falling into those regions will be discarded.

use.bamsignals: If &#x27;TRUE&#x27; the &#x27;bamsignals&#x27; package will be used for
          binning. This gives a tremendous performance increase for the
          binning step. &#x27;reads.store&#x27; and &#x27;calc.complexity&#x27; will be set
          to &#x27;FALSE&#x27; in this case.

reads.store: Set &#x27;reads.store=TRUE&#x27; to store read fragments as RData in
          folder &#x27;data&#x27; and as BED files in &#x27;BROWSERFILES/data&#x27;. This
          option will force &#x27;use.bamsignals=FALSE&#x27;.

correction.method: Correction methods to be used for the binned read
          counts. Currently only &#x27;&#x27;GC&#x27;&#x27;.

GC.BSgenome: A &#x27;BSgenome&#x27; object which contains the DNA sequence that
          is used for the GC correction.

  method: Any combination of &#x27;c(&#x27;HMM&#x27;,&#x27;dnacopy&#x27;,&#x27;edivisive&#x27;)&#x27;. Option
          &#x27;method=&#x27;HMM&#x27;&#x27; uses a Hidden Markov Model as described in
          doi:10.1186/s13059-016-0971-7 to call copy numbers. Option
          &#x27;&#x27;dnacopy&#x27;&#x27; uses &#x27;segment&#x27; from the &#x27;DNAcopy&#x27; package to call
          copy numbers similarly to the method proposed in
          doi:10.1038/nmeth.3578, which gives more robust but less
          sensitive results compared to the HMM. Option &#x27;&#x27;edivisive&#x27;&#x27;
          (DEFAULT) works like option &#x27;&#x27;dnacopy&#x27;&#x27; but uses the
          &#x27;e.divisive&#x27; function from the &#x27;ecp&#x27; package for
          segmentation.

strandseq: A logical indicating whether the data comes from Strand-seq
          experiments. If &#x27;TRUE&#x27;, both strands carry information and
          are treated separately.

       R: method-edivisive: The maximum number of random permutations
          to use in each iteration of the permutation test (see
          &#x27;e.divisive&#x27;). Increase this value to increase accuracy on
          the cost of speed.

 sig.lvl: method-edivisive: The level at which to sequentially test if
          a proposed change point is statistically significant (see
          &#x27;e.divisive&#x27;). Increase this value to find more breakpoints.

     eps: method-HMM: Convergence threshold for the Baum-Welch
          algorithm.

max.time: method-HMM: The maximum running time in seconds for the
          Baum-Welch algorithm. If this time is reached, the Baum-Welch
          will terminate after the current iteration finishes. Set
          &#x27;max.time = -1&#x27; for no limit.

max.iter: method-HMM: The maximum number of iterations for the
          Baum-Welch algorithm. Set &#x27;max.iter = -1&#x27; for no limit.

num.trials: method-HMM: The number of trials to find a fit where state
          &#x27;most.frequent.state&#x27; is most frequent. Each time, the HMM is
          seeded with different random initial values.

  states: method-HMM: A subset or all of
          &#x27;c(&quot;zero-inflation&quot;,&quot;0-somy&quot;,&quot;1-somy&quot;,&quot;2-somy&quot;,&quot;3-somy&quot;,&quot;4-somy&quot;,...)&#x27;.
          This vector defines the states that are used in the Hidden
          Markov Model. The order of the entries must not be changed.

 confint: Desired confidence interval for breakpoints. Set
          &#x27;confint=NULL&#x27; to disable confidence interval estimation.
          Confidence interval estimation will force &#x27;reads.store=TRUE&#x27;.

refine.breakpoints: A logical indicating whether breakpoints from the
          HMM should be refined with read-level information.
          &#x27;refine.breakpoints=TRUE&#x27; will force &#x27;reads.store=TRUE&#x27;.

hotspot.bandwidth: A vector the same length as &#x27;binsizes&#x27; with
          bandwidths for breakpoint hotspot detection (see &#x27;hotspotter&#x27;
          for further details). If &#x27;NULL&#x27;, the bandwidth will be chosen
          automatically as the average distance between reads.

hotspot.pval: P-value for breakpoint hotspot detection (see
          &#x27;hotspotter&#x27; for further details). Set &#x27;hotspot.pval = NULL&#x27;
          to skip hotspot detection.

cluster.plots: A logical indicating whether plots should be clustered
          by similarity.

_V_a_l_u_e:

     &#x27;NULL&#x27;

_A_u_t_h_o_r(_s):

     Aaron Taudt

_E_x_a_m_p_l_e_s:

     ## Not run:
     
     ## The following call produces plots and genome browser files for all BAM files in &quot;my-data-folder&quot;
     Aneufinder(inputfolder=&quot;my-data-folder&quot;, outputfolder=&quot;my-output-folder&quot;)
     ## End(Not run)
     

Warning in has_utility(&quot;pdfcrop&quot;) :
  pdfcrop not installed or not in PATH
sh: line 1: pdfcrop: command not found
Warning in system2(&quot;pdfcrop&quot;, shQuote(c(x, x)), stdout = if (quiet) FALSE else &quot;&quot;) :
  error in running command
sh: line 1: pdfcrop: command not found
Warning in system2(&quot;pdfcrop&quot;, shQuote(c(x, x)), stdout = if (quiet) FALSE else &quot;&quot;) :
  error in running command
sh: line 1: pdfcrop: command not found
Warning in system2(&quot;pdfcrop&quot;, shQuote(c(x, x)), stdout = if (quiet) FALSE else &quot;&quot;) :
  error in running command
sh: line 1: pdfcrop: command not found
Warning in system2(&quot;pdfcrop&quot;, shQuote(c(x, x)), stdout = if (quiet) FALSE else &quot;&quot;) :
  error in running command
sh: line 1: pdfcrop: command not found
Warning in system2(&quot;pdfcrop&quot;, shQuote(c(x, x)), stdout = if (quiet) FALSE else &quot;&quot;) :
  error in running command
sh: line 1: pdfcrop: command not found
Warning in system2(&quot;pdfcrop&quot;, shQuote(c(x, x)), stdout = if (quiet) FALSE else &quot;&quot;) :
  error in running command
Warning: ggrepel: 2 unlabeled data points (too many overlaps). Consider increasing max.overlaps
sh: line 1: pdfcrop: command not found
Warning in system2(&quot;pdfcrop&quot;, shQuote(c(x, x)), stdout = if (quiet) FALSE else &quot;&quot;) :
  error in running command
sh: line 1: pdfcrop: command not found
Warning in system2(&quot;pdfcrop&quot;, shQuote(c(x, x)), stdout = if (quiet) FALSE else &quot;&quot;) :
  error in running command
Error: processing vignette &#x27;AneuFinder.Rnw&#x27; failed with diagnostics:
unable to run pdflatex on &#x27;AneuFinder.tex&#x27;
LaTeX errors:
! LaTeX Error: File `framed.sty&#x27; not found.

Type X to quit or &lt;RETURN&gt; to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
&lt;read *&gt; 
         
l.137 \definecolor
                  {shadecolor}{RGB}{240,240,240}^^M
!  ==&gt; Fatal error occurred, no output PDF file produced!
--- failed re-building ‘AneuFinder.Rnw’

SUMMARY: processing the following file failed:
  ‘AneuFinder.Rnw’

Error: Vignette re-building failed.
Execution halted
</PRE>
</DIV></BODY>
</HTML>
