<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for dada2 on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><B>[<A href="..#D">D</A>]</B></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-06 11:00:29 -0000 (Tue, 06 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for dada2 on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the dada2 package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/dada2.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>480</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">dada2</A>&nbsp;1.29.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/dada2">landing page</A>)</SPAN><BR>Benjamin Callahan<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-05&nbsp;06:35:06&nbsp;-0000&nbsp;(Mon,&nbsp;05&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/dada2</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">b30afa0</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:44:36&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: dada2</TD></TR>
<TR><TD><B>Version</B>: 1.29.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:dada2.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings dada2_1.29.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-05 20:44:40 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-05 20:50:51 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 371.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: dada2.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:dada2.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings dada2_1.29.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/dada2.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘dada2/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘dada2’ version ‘1.29.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘dada2’ can be installed ... OK
* used C++ compiler: ‘g++ (GCC) 10.3.1’
* checking C++ specification ... NOTE
  Specified C++11: please drop specification unless essential
* checking installed package size ... NOTE
  installed size is 12.6Mb
  sub-directories of 1Mb or more:
    extdata   2.1Mb
    libs      9.8Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... NOTE
File
  LICENSE
is not mentioned in the DESCRIPTION file.
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
Unexported object imported by a &#x27;:::&#x27; call: ‘ShortRead:::.set_omp_threads’
  See the note in ?`:::` about the use of this operator.
There are ::: calls to the package&#x27;s namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘C_isACGT’ ‘matchGenera’
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... NOTE
Foreign function call to a different package:
  .Call(ShortRead:::.set_omp_threads, ...)
See chapter ‘System and foreign language interfaces’ in the ‘Writing R
Extensions’ manual.
* checking R code for possible problems ... NOTE
tax.check: warning in assignTaxonomy(sq.test, fn.tax, multi = TRUE):
  partial argument match of &#x27;multi&#x27; to &#x27;multithread&#x27;
plotComplexity: no visible binding for global variable ‘complexity’
plotErrors: no visible binding for global variable ‘Qual’
plotErrors: no visible binding for global variable ‘Observed’
plotErrors: no visible binding for global variable ‘Input’
plotErrors: no visible binding for global variable ‘Estimated’
plotErrors: no visible binding for global variable ‘Nominal’
plotQualityProfile: no visible binding for global variable ‘Cycle’
plotQualityProfile: no visible binding for global variable ‘Score’
plotQualityProfile: no visible binding for global variable ‘Count’
plotQualityProfile: no visible binding for global variable ‘Mean’
plotQualityProfile: no visible binding for global variable ‘Q25’
plotQualityProfile: no visible binding for global variable ‘Q50’
plotQualityProfile: no visible binding for global variable ‘Q75’
plotQualityProfile: no visible binding for global variable ‘Cum’
tax.check: no visible binding for global variable ‘fn.spc’
Undefined global functions or variables:
  Count Cum Cycle Estimated Input Mean Nominal Observed Q25 Q50 Q75
  Qual Score complexity fn.spc
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... NOTE
   PKG_CFLAGS set in ‘src/Makevars.win’ without any corresponding files
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
              user system elapsed
PacBioErrfun 8.806  0.075   8.898
learnErrors  6.981  0.077   5.805
mergePairs   5.414  0.056   5.478
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘dada2-intro.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 10 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/dada2.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">dada2.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL dada2
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘dada2’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (GCC) 10.3.1’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c Rmain.cpp -o Rmain.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c chimera.cpp -o chimera.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c cluster.cpp -o cluster.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c containers.cpp -o containers.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c error.cpp -o error.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c evaluate.cpp -o evaluate.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c filter.cpp -o filter.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c kmers.cpp -o kmers.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c misc.cpp -o misc.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c nwalign_endsfree.cpp -o nwalign_endsfree.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c nwalign_vectorized.cpp -o nwalign_vectorized.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c pval.cpp -o pval.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c taxonomy.cpp -o taxonomy.o
g++ -std=gnu++11 -shared -L/home/biocbuild/R/R-4.3.0/lib -L/usr/local/lib -o dada2.so RcppExports.o Rmain.o chimera.o cluster.o containers.o error.o evaluate.o filter.o kmers.o misc.o nwalign_endsfree.o nwalign_vectorized.o pval.o taxonomy.o -L/home/biocbuild/R/R-4.3.0/lib -lR
installing to /home/biocbuild/R/R-4.3.0/site-library/00LOCK-dada2/00new/dada2/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (dada2)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">dada2.Rcheck/dada2-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>PacBioErrfun</TD><TD style="text-align: right;">8.806</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">8.898</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addSpecies</TD><TD style="text-align: right;">3.567</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">3.666</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assignSpecies</TD><TD style="text-align: right;">1.668</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.671</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assignTaxonomy</TD><TD style="text-align: right;">0.313</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.325</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collapseNoMismatch</TD><TD style="text-align: right;">1.711</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">1.773</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dada</TD><TD style="text-align: right;">3.865</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">3.903</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>derepFastq</TD><TD style="text-align: right;">1.026</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.041</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fastqFilter</TD><TD style="text-align: right;">0.566</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.572</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fastqPairedFilter</TD><TD style="text-align: right;">0.934</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.945</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterAndTrim</TD><TD style="text-align: right;">1.090</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.096</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getDadaOpt</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getErrors</TD><TD style="text-align: right;">1.564</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.575</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSequences</TD><TD style="text-align: right;">0.489</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.490</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getUniques</TD><TD style="text-align: right;">0.458</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.458</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>inflateErr</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isBimera</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.091</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isBimeraDenovo</TD><TD style="text-align: right;">3.160</TD><TD style="text-align: right;">0.080</TD><TD style="text-align: right;">3.244</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isBimeraDenovoTable</TD><TD style="text-align: right;">3.570</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">3.580</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isPhiX</TD><TD style="text-align: right;">0.239</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.243</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isShiftDenovo</TD><TD style="text-align: right;">1.547</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.554</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>learnErrors</TD><TD style="text-align: right;">6.981</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">5.805</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loessErrfun</TD><TD style="text-align: right;">0.494</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.500</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeSequenceTable</TD><TD style="text-align: right;">0.885</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.885</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergePairs</TD><TD style="text-align: right;">5.414</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">5.478</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeSequenceTables</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>noqualErrfun</TD><TD style="text-align: right;">1.076</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.086</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nwalign</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nwhamming</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotComplexity</TD><TD style="text-align: right;">0.363</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.364</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotErrors</TD><TD style="text-align: right;">2.638</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.659</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotQualityProfile</TD><TD style="text-align: right;">4.359</TD><TD style="text-align: right;">0.283</TD><TD style="text-align: right;">4.595</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rc</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>removeBimeraDenovo</TD><TD style="text-align: right;">3.253</TD><TD style="text-align: right;">0.291</TD><TD style="text-align: right;">3.550</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>removePrimers</TD><TD style="text-align: right;">0.283</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.284</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqComplexity</TD><TD style="text-align: right;">0.03</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.03</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setDadaOpt</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>uniquesToFasta</TD><TD style="text-align: right;">0.091</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.091</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
