<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for systemPipeR on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><B>[<A href="..#S">S</A>]</B></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-06-05 06:03:09 -0000 (Mon, 05 Jun 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4366</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">CHECK results for systemPipeR on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the systemPipeR package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/systemPipeR.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2034</B>/2199</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">systemPipeR</A>&nbsp;2.7.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/systemPipeR">landing page</A>)</SPAN><BR>Thomas Girke<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-06-03&nbsp;10:56:11&nbsp;-0000&nbsp;(Sat,&nbsp;03&nbsp;Jun&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/systemPipeR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">34ded0c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:34:33&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc selected"><A href="./kunpeng2-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: systemPipeR</TD></TR>
<TR><TD><B>Version</B>: 2.7.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:systemPipeR.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings systemPipeR_2.7.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-06-05 01:05:04 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-06-05 01:13:10 -0000 (Mon, 05 Jun 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 485.8 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: systemPipeR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:systemPipeR.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings systemPipeR_2.7.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/systemPipeR.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (GCC) 10.3.1
    GNU Fortran (GCC) 10.3.1
* running under: openEuler 22.03 (LTS-SP1)
* using session charset: UTF-8
* checking for file ‘systemPipeR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘systemPipeR’ version ‘2.7.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  inst/extdata/.batchtools.conf.R
  inst/extdata/.batchtools.conf.R.sge
  inst/extdata/.batchtools.conf.R.torque
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘systemPipeR’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is  6.9Mb
  sub-directories of 1Mb or more:
    R             1.6Mb
    extdata       2.9Mb
    htmlwidgets   2.0Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... NOTE
Problems with news in ‘NEWS’:
  Cannot process chunk/lines:
    OVERVIEW
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.cmd2cwl2: no visible global function definition for ‘ave’
.getPreprotext: no visible binding for global variable ‘.’
Undefined global functions or variables:
  . ave
Consider adding
  importFrom(&quot;stats&quot;, &quot;ave&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
               user system elapsed
genFeatures  26.692  0.679  27.423
run_DESeq2   12.924  0.080  13.033
loadWorkflow  7.175  0.091   7.280
renderLogs    5.219  0.362   5.625
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘systemPipeR.Rmd’ using ‘UTF-8’... OK
  ‘systemPipeR_workflows.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/systemPipeR.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">systemPipeR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL systemPipeR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘systemPipeR’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (systemPipeR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">systemPipeR.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(systemPipeR)
Loading required package: Rsamtools
Loading required package: GenomeInfoDb
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: ShortRead
Loading required package: BiocParallel
Loading required package: GenomicAlignments
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

&gt; 
&gt; test_check(&quot;systemPipeR&quot;)
[1] &quot;Generated /tmp/Rtmpzfzw3x/rnaseq directory. Next run in rnaseq directory, the R code from *.Rmd template interactively. Alternatively, workflows can be exectued with a single command as instructed in the vignette.&quot;
[1] &quot;Generated /tmp/Rtmpzfzw3x/rnaseq2 directory. Next run in rnaseq directory, the R code from *.Rmd template interactively. Alternatively, workflows can be exectued with a single command as instructed in the vignette.&quot;
Creating directory &#x27;/tmp/Rtmpzfzw3x/rnaseq2/.SPRproject&#x27;
Creating file &#x27;/tmp/Rtmpzfzw3x/rnaseq2/.SPRproject/SYSargsList.yml&#x27;
Running Step:  example 
Running Session: Management 
  |                                                                              |                                                                      |   0%  |                                                                              |=======================                                               |  33%  |                                                                              |===============================================                       |  67%  |                                                                              |======================================================================| 100%
---- Summary ---- 
   Targets Total_Files Existing_Files Missing_Files example
M1      M1           1              1             0 Success
M2      M2           1              1             0 Success
M3      M3           1              1             0 Success

Step Status:  Success 
Done with workflow running, now consider rendering logs &amp; reports
To render logs, run:    sal &lt;- renderLogs(sal)
From command-line:      Rscript -e &quot;sal = systemPipeR::SPRproject(resume = TRUE); sal = systemPipeR::renderLogs(sal)&quot;
To render reports, run: sal &lt;- renderReport(sal)
From command-line:      Rscript -e &quot;sal= s ystemPipeR::SPRproject(resume = TRUE); sal = systemPipeR::renderReport(sal)&quot;
This message is displayed once per R session
Running Step:  newStep 
Running Session: Management 
  |                                                                              |                                                                      |   0%  |                                                                              |=======================                                               |  33%  |                                                                              |===============================================                       |  67%  |                                                                              |======================================================================| 100%
---- Summary ---- 
   Targets Total_Files Existing_Files Missing_Files example
M1      M1           1              1             0 Success
M2      M2           1              1             0 Success
M3      M3           1              1             0 Success

Step Status:  Success 
Running Step:  R_code 
Running Session: Management 
  |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%
Step Status:  Warning 
[1] &quot;Generated /tmp/Rtmpzfzw3x/newtest directory. Next run in new directory, the R code from *.Rmd template interactively. Alternatively, workflows can be exectued with a single command as instructed in the vignette.&quot;
Creating directory &#x27;/tmp/Rtmpzfzw3x/newtest/.SPRproject&#x27;
Creating file &#x27;/tmp/Rtmpzfzw3x/newtest/.SPRproject/SYSargsList.yml&#x27;
Reading Rmd file
Now importing step &#x27;load_library&#x27; 
Now importing step &#x27;export_iris&#x27; 
Now importing step &#x27;gzip&#x27; 
Now importing step &#x27;gunzip&#x27; 
Now importing step &#x27;stats&#x27; 
Now back up current Rmd file as template for `renderReport`
Template for renderReport is stored at 
 /tmp/Rtmpzfzw3x/newtest/.SPRproject/workflow_template.Rmd 
 Edit this file manually is not recommended 
import  done
Instance of &#x27;LineWise&#x27;
    Code Chunk length: 1
mapply(function(x, y) write.csv(x, y), split(iris, factor(iris$Species)), file.path(&quot;results&quot;, paste0(names(split(iris, factor(iris$Species))), &quot;.csv&quot;)))
load_library
    5 + 5
load_library
    5 + 5
    66 + 55
Running Step:  load_library 
Running Session: Management 
  |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%
Step Status:  Success 
Running Step:  export_iris 
Running Session: Management 
  |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%
Step Status:  Success 
Skipping Step:  gzip 
Skipping Step:  gunzip 
Skipping Step:  stats 
[ FAIL 0 | WARN 0 | SKIP 10 | PASS 47 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• On Bioconductor (10)

[ FAIL 0 | WARN 0 | SKIP 10 | PASS 47 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 16.651   1.536  18.896 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">systemPipeR.Rcheck/systemPipeR-Ex.timings</SPAN></P>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>EnvModules-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GOHyperGAll</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>INTERSECTset-class</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>LineWise-class</TD><TD style="text-align: right;">0.206</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">0.216</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ParamFiles</TD><TD style="text-align: right;">0.992</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.998</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ParamFiles2</TD><TD style="text-align: right;">0.175</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.175</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SPRproject</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SYSargs-class</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SYSargs2-class</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SYSargsList-class</TD><TD style="text-align: right;">0.561</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.580</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SYSargsList</TD><TD style="text-align: right;">0.114</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.126</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>VENNset-class</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>alignStats</TD><TD style="text-align: right;">1.590</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">1.635</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>catDB-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>catmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>check.output</TD><TD style="text-align: right;">1.702</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.722</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>clusterRun</TD><TD style="text-align: right;">1.633</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.644</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>config.param</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>configWF</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countRangeset</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.057</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createParamFiles</TD><TD style="text-align: right;">0.212</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.213</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cwlFilesUpdate</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>evalCode</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featureCoverage</TD><TD style="text-align: right;">1.609</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.613</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featuretypeCounts</TD><TD style="text-align: right;">1.619</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">1.651</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterDEGs</TD><TD style="text-align: right;">2.112</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">2.146</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterVars</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genFeatures</TD><TD style="text-align: right;">26.692</TD><TD style="text-align: right;"> 0.679</TD><TD style="text-align: right;">27.423</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>importWF</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadWorkflow</TD><TD style="text-align: right;">7.175</TD><TD style="text-align: right;">0.091</TD><TD style="text-align: right;">7.280</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeBamByFactor</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>moduleload</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>olBarplot</TD><TD style="text-align: right;">0.556</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.566</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>olRanges</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.140</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>output_update</TD><TD style="text-align: right;">1.814</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.840</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>overLapper</TD><TD style="text-align: right;">0.454</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.463</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotfeatureCoverage</TD><TD style="text-align: right;">1.569</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.571</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotfeaturetypeCounts</TD><TD style="text-align: right;">1.789</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.799</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>predORF</TD><TD style="text-align: right;">0.240</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.244</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>preprocessReads</TD><TD style="text-align: right;">0.557</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.565</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readComp</TD><TD style="text-align: right;">1.803</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.817</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>renderLogs</TD><TD style="text-align: right;">5.219</TD><TD style="text-align: right;">0.362</TD><TD style="text-align: right;">5.625</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>renderReport</TD><TD style="text-align: right;">1.881</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">1.981</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>returnRPKM</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runCommandline</TD><TD style="text-align: right;">1.610</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.637</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runDiff</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runWF</TD><TD style="text-align: right;">0.118</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.127</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_DESeq2</TD><TD style="text-align: right;">12.924</TD><TD style="text-align: right;"> 0.080</TD><TD style="text-align: right;">13.033</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_edgeR</TD><TD style="text-align: right;">1.641</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.647</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sal2bash</TD><TD style="text-align: right;">0.615</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.629</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sal2rmd</TD><TD style="text-align: right;">0.846</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.877</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scaleRanges</TD><TD style="text-align: right;">0.233</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.237</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seeFastq</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>showDF</TD><TD style="text-align: right;">0.050</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subsetWF</TD><TD style="text-align: right;">1.572</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.576</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>symLink2bam</TD><TD style="text-align: right;">1.819</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.823</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sysargs</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>systemArgs</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>targets.as.df</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>trimbatch</TD><TD style="text-align: right;">0.347</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.357</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tryCMD</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tryPath</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>variantReport</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>vennPlot</TD><TD style="text-align: right;">0.475</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.489</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeTargets</TD><TD style="text-align: right;">0.117</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.123</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeTargetsRef</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeTargetsout</TD><TD style="text-align: right;">1.844</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.860</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>write_SYSargsList</TD><TD style="text-align: right;">0.139</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.140</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
