<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - BUILD results for SimFFPE on kunpeng2</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><B>[<A href="..#S">S</A>]</B></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-25 05:46:01 -0000 (Thu, 25 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng2"><TD><B><A href="../kunpeng2-NodeInfo.html"><B>kunpeng2</B></A></B></TD><TD>Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng2-R-instpkgs.html">4223</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng2">BUILD results for SimFFPE on kunpeng2</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the SimFFPE package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/SimFFPE.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1895</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc selected">BUILD</TD><TD class="STAGE checksrc">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">SimFFPE</A>&nbsp;1.13.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/SimFFPE">landing page</A>)</SPAN><BR>Lanying Wei<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-24&nbsp;15:28:50&nbsp;-0000&nbsp;(Wed,&nbsp;24&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/SimFFPE</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">ea3cf0a</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:15:01&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng2 selected"><B>kunpeng2</B></TD><TD class="kunpeng2"><SPAN style="font-size: smaller;">Linux&nbsp;(openEuler&nbsp;22.03&nbsp;LTS-SP1)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng2 install"><A href="./kunpeng2-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 buildsrc selected"><A href="./kunpeng2-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng2 checksrc"><SPAN class="glyph skipped">skipped</SPAN></TD><TD class="status kunpeng2"></TD><TD class="status kunpeng2"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng2 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: SimFFPE</TD></TR>
<TR><TD><B>Version</B>: 1.13.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data SimFFPE</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-25 01:44:24 -0000 (Thu, 25 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-25 01:46:33 -0000 (Thu, 25 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 129.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>PackageFile</B>: None</TD></TR>
<TR><TD><B>PackageFileSize</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng2 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD build --keep-empty-dirs --no-resave-data SimFFPE
###
##############################################################################
##############################################################################


* checking for file ‘SimFFPE/DESCRIPTION’ ... OK
* preparing ‘SimFFPE’:
* checking DESCRIPTION meta-information ... OK
* installing the package to process help pages
* saving partial Rd database
* creating vignettes ... ERROR
--- re-building ‘SimFFPE.Rnw’ using Sweave
Loading required package: Biostrings
Loading required package: BiocGenerics

Attaching package: ‘BiocGenerics’

The following objects are masked from ‘package:stats’:

    IQR, mad, sd, var, xtabs

The following objects are masked from ‘package:base’:

    Filter, Find, Map, Position, Reduce,
    anyDuplicated, aperm, append, as.data.frame,
    basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl,
    intersect, is.unsorted, lapply, mapply, match,
    mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff,
    sort, table, tapply, union, unique, unsplit,
    which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: ‘S4Vectors’

The following object is masked from ‘package:utils’:

    findMatches

The following objects are masked from ‘package:base’:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: XVector
Loading required package: GenomeInfoDb

Attaching package: ‘Biostrings’

The following object is masked from ‘package:base’:

    strsplit

BAM filesize smaller than 1 gigabytes, calculate Phred quality score profile based on all reads.
Get reads from BAM file in 10 regions.
Using 1 thread(s).
Using 2 thread(s) for simulation.
Simulating FFPE reads on the chromosome 1...
Generated 474 adjacent chimeric reads on chromosome 1.
Generated 1136 distant chimeric reads on chromosome 1.
Generated 14378 normal reads on chromosome 1.
Simulating FFPE reads on the chromosome 2...
Generated 428 adjacent chimeric reads on chromosome 2.
Generated 1204 distant chimeric reads on chromosome 2.
Generated 14384 normal reads on chromosome 2.
Simulating FFPE reads on the chromosome 3...
Generated 468 adjacent chimeric reads on chromosome 3.
Generated 1300 distant chimeric reads on chromosome 3.
Generated 14390 normal reads on chromosome 3.
In totoal 1370 adjacent chimeric reads, 3640 distant chimeric reads, 43152 normal reads were generated.
Alltogether 48162 reads were generated. 5010 reads (10.4%, adjusted by chimericProp) are artifact chimeric reads.
Of all artifact chimeric reads, 1370 reads (27.35%, adjusted by sameChrProp * adjChimProp) are adjacent chimeric reads.
Simulation done.
Using 1 thread(s) for simulation.
Simulating FFPE reads on the chromosome 1...
Generated 156 adjacent chimeric reads on chromosome 1.
Generated 532 distant chimeric reads on chromosome 1.
Generated 4774 normal reads on chromosome 1.
Simulating FFPE reads on the chromosome 2...
Generated 144 adjacent chimeric reads on chromosome 2.
Generated 490 distant chimeric reads on chromosome 2.
Generated 4770 normal reads on chromosome 2.
In totoal 300 adjacent chimeric reads, 1022 distant chimeric reads, 9544 normal reads were generated.
Alltogether 10866 reads were generated. 1322 reads (12.17%, adjusted by chimericProp) are artifact chimeric reads.
Of all artifact chimeric reads, 300 reads (22.69%, adjusted by sameChrProp * adjChimProp) are adjacent chimeric reads.
Simulation done.
Using 1 thread(s) for simulation.
Simulating FFPE reads on the chromosome 1...
Generated 68 adjacent chimeric reads on chromosome 1.
Generated 40 distant chimeric reads on chromosome 1.
Generated 2706 normal reads on chromosome 1.
Simulating FFPE reads on the chromosome 3...
Generated 64 adjacent chimeric reads on chromosome 3.
Generated 116 distant chimeric reads on chromosome 3.
Generated 2446 normal reads on chromosome 3.
Simulating FFPE reads on the chromosome 8...
Generated 46 adjacent chimeric reads on chromosome 8.
Generated 80 distant chimeric reads on chromosome 8.
Generated 2028 normal reads on chromosome 8.
In totoal 178 adjacent chimeric reads, 236 distant chimeric reads, 7180 normal reads were generated.
Alltogether 7594 reads were generated. 414 reads (5.45%, adjusted by chimericProp) are artifact chimeric reads.
Of all artifact chimeric reads, 178 reads (43%, adjusted by sameChrProp * adjChimProp) are adjacent chimeric reads.
Simulation done.
Error: processing vignette &#x27;SimFFPE.Rnw&#x27; failed with diagnostics:
unable to run pdflatex on &#x27;SimFFPE.tex&#x27;
LaTeX errors:
! LaTeX Error: File `framed.sty&#x27; not found.

Type X to quit or &lt;RETURN&gt; to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
&lt;read *&gt; 
         
l.137 \definecolor
                  {shadecolor}{RGB}{240,240,240}^^M
!  ==&gt; Fatal error occurred, no output PDF file produced!
--- failed re-building ‘SimFFPE.Rnw’

SUMMARY: processing the following file failed:
  ‘SimFFPE.Rnw’

Error: Vignette re-building failed.
Execution halted
</PRE>
</DIV></BODY>
</HTML>
